%TEMPLATE_COMMENTS
\nonstopmode
\documentclass[10pt,letterpaper]{article}
\usepackage[top=1in,left=1in,right=1in,bottom=1in]{geometry}
%\usepackage[top=0.85in,left=2.75in,footskip=0.75in]{geometry}

% long tables
\usepackage{longtable}

% amsmath and amssymb packages, useful for mathematical formulas and symbols
\usepackage{amsmath,amssymb, mathtools}
\newcommand{\one}[1]{\mathop{{}\mathbf{1}}\nolimits_{#1}}

% Use adjustwidth environment to exceed column width (see example table in text)
\usepackage{changepage}

% textcomp package and marvosym package for additional characters
\usepackage{textcomp,marvosym}

% cite package, to clean up citations in the main text. Do not remove.
\usepackage{cite}

% Use nameref to cite supporting information files (see Supporting Information section for more info)
\usepackage{nameref,hyperref}

% line numbers
\usepackage[right]{lineno}

% ligatures disabled
\usepackage[nopatch=eqnum]{microtype}
\DisableLigatures[f]{encoding = *, family = * }


% color can be used to apply background shading to table cells only
\usepackage[table]{xcolor}

% array package and thick rules for tables
\usepackage{array}

% easier turning on/off of includegraphics
\usepackage{ifthen}

% create "+" rule type for thick vertical lines
\newcolumntype{+}{!{\vrule width 2pt}}

% create \thickcline for thick horizontal lines of variable length
\newlength\savedwidth
\newcommand\thickcline[1]{%
  \noalign{\global\savedwidth\arrayrulewidth\global\arrayrulewidth 2pt}%
  \cline{#1}%
  \noalign{\vskip\arrayrulewidth}%
  \noalign{\global\arrayrulewidth\savedwidth}%
}

% \thickhline command for thick horizontal lines that span the table
\newcommand\thickhline{\noalign{\global\savedwidth\arrayrulewidth\global\arrayrulewidth 2pt}%
\hline
\noalign{\global\arrayrulewidth\savedwidth}}


% Remove comment for double spacing
%\usepackage{setspace} 
%\doublespacing

%TEXT_LAYOUT

% Bold the 'Figure #' in the caption and separate it from the title/caption with a period
% Captions will be left justified
\usepackage[aboveskip=1pt,labelfont=bf,labelsep=period,justification=raggedright,singlelinecheck=off]{caption}
\renewcommand{\figurename}{Fig}


% Use the PLoS provided BiBTeX style
\bibliographystyle{plos2015}

% Remove brackets from numbering in List of References
\makeatletter
\renewcommand{\@biblabel}[1]{\quad#1.}
\makeatother

%HEADER_FOOTER

%% Include all macros below

\newcommand{\lorem}{{\bf LOREM}}
\newcommand{\ipsum}{{\bf IPSUM}}
\newcommand{\MI}{M^{\text{\text{obs}}}}
\newboolean{includefigs}
\setboolean{includefigs}{true} % true/false


%% END MACROS SECTION

% Remove this for final submission, do not include graphics
\usepackage{graphicx} % Required for inserting images
%\usepackage{biblatex} %Imports biblatex package
%\addbibresource{references.bib} %Import the bibliography file

\usepackage{datatool}
\DTLsetseparator{,}% Set the separator between the columns.
% todo add statistics file directly into this document
\DTLloaddbtex{\mydata}{../output/statistics.dbtex}
\newcommand{\var}[1]{\DTLfetch{\mydata}{labels}{#1}{vals}}

\title{Quantifying prevalence and risk factors of HIV multiple infection in Uganda from population-based deep-sequence data}

\begin{document}
%TITLE_VPSACE

% Title must be 250 characters or less.
\begin{flushleft}

{\Large\textbf\newline{Quantifying prevalence and risk factors of HIV multiple infection in Uganda from population-based deep-sequence data} 
% Please use "sentence case" for title and headings (capitalize only the first word in a title (or heading), the first word in a subtitle (or subheading), and any proper nouns).
}
\newline
% Insert author names, affiliations and corresponding author email (do not include titles, positions, or degrees).
\\ 
Michael A. Martin\textsuperscript{1,$\dagger$},
Andrea Brizzi\textsuperscript{2},
Xiaoyue Xi\textsuperscript{2,3},
Ronald Moses Galiwango\textsuperscript{4},
Sikhulile Moyo\textsuperscript{5,6},
Deogratius Ssemwanga\textsuperscript{7,8}
Alexandra Blenkinsop\textsuperscript{2},
Andrew D. Redd\textsuperscript{9,10,11},
Lucie Abeler-Dörner\textsuperscript{12},
Christophe Fraser\textsuperscript{12},
Steven J. Reynolds\textsuperscript{4,9,10},
Thomas C. Quinn\textsuperscript{4,9,10},
Joseph Kagaayi\textsuperscript{4,13},
David Bonsall\textsuperscript{14},
David Serwadda\textsuperscript{4},
Gertrude Nakigozi\textsuperscript{4},
Godfrey Kigozi\textsuperscript{4},
M. Kate Grabowski\textsuperscript{1,4,15,$\dagger$},
Oliver Ratmann\textsuperscript{2,$\dagger$},
with the PANGEA-HIV Consortium and the Rakai Health Sciences Program
\\
\bigskip
\textbf{1} Department of Pathology, Johns Hopkins School of Medicine, Baltimore, MD, USA
\\
\textbf{2} Department of Mathematics, Imperial College London, London, United Kingdom
\\
\textbf{3} Medical Research Council Biostatistics Unit, University of Cambridge, Cambridge, UK \\

\textbf{4} Rakai Health Sciences Program, Kalisizo, Uganda \\

\textbf{5} Botswana Harvard AIDS Institute Partnership, Botswana Harvard HIV Reference Laboratory, Gaborone, Botswana \\

\textbf{6} Harvard T.H. Chan School of Public Health, Boston, MA, USA \\

\textbf{7} Medical Research Council/Uganda Virus Research Institute and London School of Hygiene and Tropical Medicine Uganda Research Unit, Entebbe, Uganda \\

\textbf{8} Uganda Virus Research Institute, Entebbe, Uganda \\

\textbf{9} Department of Medicine, Johns Hopkins School of Medicine, Baltimore, MD, USA
\\
\textbf{10} Division of Intramural Research, National Institute of Allergy and Infectious Diseases, National Institutes of Health, Bethesda, MD, USA
\\
\textbf{11} Institute of Infectious Disease and Molecular Medicine, University of Cape Town, Cape Town, South Africa
\\
\textbf{12} Pandemic Sciences Institute, Nuffield Department of Medicine, University of Oxford, Oxford, UK \\

\textbf{13} Makerere University School of Public Health, Kampala, Uganda \\

\textbf{14} Wellcome Centre for Human Genetics, Nuffield Department of Medicine, University of Oxford, Oxford, UK \\

\textbf{15} Department of Epidemiology, Johns Hopkins Bloomberg School of Public Health, Baltimore, MD, USA \\

\bigskip

% Use the asterisk to denote corresponding authorship and provide email address in note below.
$\dagger$ Corresponding authors mmart108@jhmi.edu, mgrabow2@jhu.edu, oliver.ratmann@imperial.ac.uk 
\end{flushleft}


% Please keep the abstract below 300 words
\section*{Abstract}
People living with HIV can acquire secondary infections through a process called superinfection, giving rise to simultaneous infection with genetically distinct variants (multiple infection). Multiple infection provides the necessary conditions for the generation of novel recombinant forms of HIV and may worsen clinical outcomes and increase the rate of transmission to HIV seronegative sexual partners. To date, studies of HIV multiple infection have relied on insensitive bulk-sequencing, labor intensive single genome amplification protocols, or deep-sequencing of short genome regions. Here, we identified multiple infections in whole-genome or near whole-genome HIV RNA deep-sequence data generated from plasma samples of \var{empirical_age_sex_comm_n_participant_visits} people living with viremic HIV who participated in the population-based Rakai Community Cohort Study (RCCS). We estimated individual- and population-level probabilities of being multiply infected and assessed epidemiological risk factors using the novel Bayesian deep-phylogenetic multiple infection model (\textit{deep-phyloMI}) which accounts for bias due to partial sequencing success and false-negative and false-positive detection rates. We estimated that between \var{min_survey_year} and \var{max_survey_year}, \var{empirical_age_sex_comm_fit_wprob_mi_median_percent}\% (95\% highest posterior density interval (HPD) \var{empirical_age_sex_comm_fit_wprob_mi_lower_percent}\% - \var{empirical_age_sex_comm_fit_wprob_mi_upper_percent}\%) of RCCS participants with viremic HIV multiple infection at time of sampling. Participants living in high-HIV prevalence communities along Lake Victoria were \var{empirical_age_sex_comm_fit_pr_mi_comm_type == 'fishing':comm_type=='inland'_median}-fold (95\% HPD \var{empirical_age_sex_comm_fit_pr_mi_comm_type == 'fishing':comm_type=='inland'_lower} - \var{empirical_age_sex_comm_fit_pr_mi_comm_type == 'fishing':comm_type=='inland'_upper}) more likely to harbor a multiple infection compared to individuals in lower prevalence neighboring communities. This work introduces a high-throughput surveillance framework for identifying people with multiple HIV infections and quantifying population-level prevalence and risk factors of multiple infection for clinical and epidemiological investigations. 
%
%improves upon the statistical framework used to study the prevalence of HIV multiple infections and clarifies our understanding of the risk of multiple infections in a high-burden region. These findings may help inform targeted public health interventions to reduce the risk of multiple infection acquisition. 

% Please keep the Author Summary between 150 and 200 words
% Use first person. PLOS ONE authors please skip this step. 
% Author Summary not valid for PLOS ONE submissions.   
\section*{Author summary}
HIV exists as a population of genetically distinct viral variants among people living with HIV. People living with HIV can be infected with genetically distinct variants. Identification of these mixed infections requires generating viral genomic data from people living with HIV. In the past, the approaches used to identify multiple infections from viral genomic data have had poor sensitivity or required labor intensive protocols that are prohibitive in application to large data sets. Prior work has also only utilized data generated from small portions of the viral genome and the statistical procedures used to generate population-level estimates from sequencing data generated from individual infections has not accounted for incomplete sampling of the within-host viral population or sources of sequencing error, which may confound multiple infection estimates. Here, we develop a statistical model that addresses these limitations and allows for the identification of multiple infections and the estimation of population-level risk of multiple infection from deep-sequence data. We fit this model to population-based HIV genomic data from people living with HIV in southern Uganda and estimate that approximately \var{empirical_age_sex_comm_fit_wprob_mi_median_percent_int}\% of viremic participants harbor a multiple infection at a given point in time. We show that the prevalence of multiple infections is higher in key populations with high HIV prevalence. These findings inform our understanding of the sexual risk networks that give rise to multiple infections and aid in efforts to model HIV epidemiological dynamics and evolution during a period of incidence declines and shifting transmission dynamics across Eastern and Southern Africa. 


\linenumbers

%----------------------------------------------------------------------------------------
%   INTRODUCTION
%----------------------------------------------------------------------------------------
\section{Introduction}
Simultaneous infection with multiple distinct variants of human immunodeficiency virus (HIV) can occur through a process called superinfection following secondary exposure to infected bodily fluids \cite{redd2013}. Following acquisition, infecting variants are shaped by within-individual evolutionary processes and can either stably coexist or undergo competitive exclusion \cite{redd2012, wertheim2018}. Superinfection of PLHIV has important implications for the evolution, pathogenesis, and spread of HIV. Specifically, it provides the necessary conditions for the generation of novel recombinant viruses \cite{fang2004, streeck2008}, which fuels diversification of the circulating viral population \cite{ramirez2008, kim2024}, complicating vaccine development efforts through the generation of novel epitopes \cite{ritchie2014, corey2010} and potentially leads to the evolution of more transmissible viral genotypes \cite{kiwanuka2009}. Acquisition of superinfections may also increase the breadth and strength of the antibody response to HIV infection \cite{powell2010, cortez2012, krebs2019}, potentially aiding in the identification of broadly neutralizing antibodies \cite{sok018}. Finally, multiple infections may themselves lead to faster disease progression \cite{gottlieb2004, smith2004, ronen2014} and higher viral load \cite{ronen2014, smith2004}, thereby potentially also increasing the risk of onward transmission \cite{quinn2000, fraser2007}. While the availability of viral genome sequence data has allowed for the identification of HIV multiple infections across a range of epidemiological contexts \cite{yuan2024}, prevalence estimates have generally been based on relatively small samples sizes with only partial genome data. Here, we identify HIV multiple infections using within-host deep-sequence phylogenetic trees inferred across the genome from a population-based surveillance cohort. \par

To date, viral sequence-based methods to identify HIV multiple infections have generally relied on one of three approaches. First, bulk sequencing (e.g. Sanger  sequencing or consensus sequence estimation from deep-sequence data) can reveal instances where the majority viral variant changes between baseline and follow-up visits under longitudinal sampling or cases where the within-person viral population at a specific visit harbors abnormal levels of diversity \cite{smith2004, cornelissen2007, vanderkuyl2009, chaudron2022}. While this approach proved useful prior to the availability of deep-sequencing technologies, it has a sensitivity of only $\sim5\%$ to detect variants present in $\leq20\%$ of the within-host viral population \cite{rachinger2010}. Alternatively, single genome amplification (SGA) relies on serial dilutions to isolate a single molecule of transcribed viral cDNA prior to amplification and sequencing \cite{sheward2015, ssemwanga2018, woodson2019}. This approach is more sensitive in detecting minor variants than bulk sequencing and was considered the ``gold standard'' \cite{pacold2010}, but is labor intensive and difficult to apply at scale. Amplicon deep-sequencing of discrete regions of the HIV genome is able to achieve high sensitivity while being highly scalable to large sets of samples and has therefore been broadly applied to study multiple infections in larger studies \cite{pacold2010, redd2011, redd2012, redd2014, ronen2013}. \par

Despite advancements in viral sequence-based identification of HIV multiple infections, existing approaches share shortcomings that hinder the interpretation of the results they generate. Critically, all of these methods rely on sequence data generated from only a subset of the genome, due in part to historical challenges in generating whole-genome HIV sequence data. For example, general population-based studies in Rakai, Uganda have previously utilized sequence data from 390 base pairs (bp) and 324 bp of the p24 (\textit{gag}) and gp41 (\textit{env}) regions, representing only 7.3\% of the HIV genome. This inherently limits sensitivity to identify multiple infection with viral variants that are highly related within these short regions. Analysis of \textit{gag} sequence data sampled from high-risk Kenyan women revealed cases of superinfection that were unidentified when querying only the \textit{env} region \cite{piantadosi2008}. Further, limited consideration has been given to the fact that factors that affect sequencing success of biological samples \cite{bonsall2020} may also affect the detection probability of multiple viral variants and may therefore confound prevalence estimates and assessment of multiple infection risk factors. Finally, existing methods generally use binary categorization of samples as either multiply or singly infected. They do not quantify uncertainty in individual-level assignments and do not account for this uncertainty when estimating population-level prevalence. With the advent of approaches that can generate near whole-genome HIV deep-sequence data \cite{gall2012, bonsall2020}, there is a need for statistical approaches that can integrate data from across the genome to robustly identify multiple infections while accounting for the various sources of bias that can obscure the underlying biological signal. \par

Here, we identify individuals that are likely to have multiple HIV multiple infection at the time of sampling, provide minimum estimates of the prevalence of HIV multiple infections in Rakai, Uganda between \var{min_survey_date} and \var{max_survey_date}, and characterize risk factors for harboring a multiple infection based on HIV RNA deep-sequence data obtained from plasma samples of \var{empirical_age_sex_comm_n_participant_visits} people living with viremic HIV aged 15-49 who participated in the longitudinal, population-based Rakai Community Cohort Study (RCCS) \cite{grabowski2017, chang2016}. These estimates reflect multiple infections present at time of sampling in plasma and, because infecting variants may be lost over time due to within-host evolutionary processes \cite{redd2012, wertheim2018}, should be interpreted as the minimum prevalence of people who have ever been multiply infected. Rakai District is located in south-central Uganda, East Africa, bordering Lake Victoria, and is one of the areas with highest HIV-prevalence globally \cite{dwyer-lindgren2019}. To support these inferences, we developed a novel Bayesian statistical model to identify multiple infections using within-host phylogenetic trees inferred from deep-sequence data generated from across the HIV genome, which we call the deep-phylo multiple infection model (\textit{deep-phyloMI}). \textit{Phyloscanner} \cite{wymant2017}, which analyzes within-host pathogen diversity from deep-sequencing reads, was used to infer within-host phylogenetic trees across the HIV genome, remove contaminant sequences, and identify regions of the genome with evidence of multiple infecting variants. Our model simultaneously estimates individual- and population-level risks of harboring a multiple infection from processed \textit{phyloscanner} output after accounting for incomplete sequencing of the viral population within a sample and false-negative and false-positive rates of multiple variant identification. We validated model performance on simulated data and used it to identify multiple infections in RCCS participants over a period of declining incidence and rapidly shifting transmission dynamics~\cite{grabowski2017, monod2024}.
\par



%----------------------------------------------------------------------------------------
%   METHODS
%----------------------------------------------------------------------------------------
\section{Materials and methods}
\label{Materials and methods}
\subsection{Study design and participants}
\label{sec:rccs_methods}
The RCCS conducts population-based surveys every 18-24 months in agrarian, semi-urban trading, and Lake Victoria fishing communities in southern Uganda. Data in this study were collected over \var{n_rounds_char} RCCS survey rounds conducted between \var{min_survey_date} and \var{max_survey_date}. As survey rounds occurred over more than a year, we herein refer to them by the median interview date. Communities that participated in the RCCS were categorized based on their geographic setting and primary economic activity (inland communities: agrarian / trading, Lake Victoria communities: fishing). These communities differ considerably in their HIV burden (HIV prevalence of $\sim$14\% [agrarian], $\sim$17\% [trading], and $\sim$42\% [fishing]) \cite{chang2016}. At each survey round, households were censused and all residents aged 15-49 who were able to provide consent (assent for those under 18) were invited to participate in a survey. Survey participants were eligible to participate exactly once in each survey round (``participant-visits''). As part of the survey, participants completed a detailed structured sociodemographic, behavioral, and health questionnaire. Specifically, participants were asked to self-report their sex, age, residency status (e.g. recent migration into a community), circumcision status (among males), occupation, occupation of sex partners in the year prior to the survey, and number of lifetime sex partners. As HIV is more prevalent among female sex and bar/restaurant workers \cite{dambach2020, popoola2024}, we generated a composite variable indicating reported sex or bar/restaurant work among women and sex with a sex or bar/restaurant worker among men to determine if these individuals were at higher risk of being multiply infected. \par

To account for the fact that the number of lifetime sex partners increases over the lifespan, we calculated the mean number of lifetime sex partners within population strata ($s$) defined by HIV serostatus, sex, age category in five year bins, and community type (inland / fishing) ($\bar{P_s}$) to allow for standardization of the observed responses. Responses of no lifetime sex partners were treated as missing data as HIV transmission in this setting is predominantly heterosexual \cite{ugandaAIDS2012} and we therefore expected these individuals to have had at least one sexual encounter in order to acquire HIV, although we cannot rule-out perinatal transmission with available data. When calculating $\bar{P_s}$ missing data was imputed to the mean value of a lognormal distribution fit to all numeric responses of $\ge1$ lifetime sex partner within strata defined by HIV serostatus, sex, age category, and community type. Additionally, some RCCS participants provided categorical responses (``1-2'' or ``3+'' lifetime sex partners). To calculate $\bar{P_s}$, we first imputed these values to a numeric response. Responses of ``1-2'' were imputed to the mean response among PLHIV reporting either one or two lifetime sex partners within strata. Similarly, responses of ``3+'' were imputed to the mean value of a lognormal distribution fit to all numeric responses of $\ge3$ lifetime partners within strata as above. 

In addition to completing the survey questionnaire, participants provided venous blood samples for HIV testing, viral load quantification, and viral deep sequencing. HIV serostatus was evaluated using a validated rapid test algorithm \cite{kagulire2011}. HIV viral load quantification was conducted using the Abbott real-time m2000 assay (Abbott Laboratories). All participants provided written informed consent for the study. Written assent and parental consent were obtained for participants less than 18 years of age. The RCCS is administered by the Rakai Health Sciences Program (RHSP) and has received ethical approval from the Uganda Virus Research Institute’s Research and Ethics Committee (GC/127/08/12/137), the Uganda National Council for Science and Technology (HS450), and the Johns Hopkins School of Medicine (IRB00217467).  

\subsection{HIV deep sequencing and bioinformatic processing}
HIV RNA deep-sequence data from plasma samples contributed by RCCS participants was generated through the Phylogenetics and Networks for Generalized HIV Epidemics in Africa consortium (PANGEA-HIV)~\cite{pillay2015,abeler-dorner2019}. The study sample included RCCS participants with HIV who were viremic ($\ge1,000$ copies/mL) at one of their study visits between \var{min_survey_date} and \var{max_survey_date}. To avoid biasing our inferences, for individuals that participated in multiple survey rounds we used only the data from the sample with the highest genome coverage or the highest viral load in the case of ties in our analyses of multiple infections. The study sample was further restricted to individuals in putative transmission networks and excluded individuals for who another phylogenetically close individual could not be identified over the entire study period~\cite{monod2024}. All available sequence data for individuals in putative transmission networks was included in phylogenetic analyses. \par

Deep-sequencing was performed with two protocols (\nameref{seq_tech_table}), as previously described~\cite{monod2024}. Briefly, for sequence data generated through the amplicon protocol, viral RNA was extracted from plasma samples on the QIAsymphony SP workstation with the QIAsymphony DSP Virus/Pathogen Kit. cDNA was generated through one-step reverse transcription PCR protocol using universal HIV-1 primers designed to generate four overlapping amplicons across the HIV-1 genome~\cite{gall2012}. Deep-sequencing was conducted at the Wellcome Trust Sanger Institute core facility using the Illumina MiSeq and HiSeq platforms. To generate sequence data using the bait-capture protocol viral RNA was similarly extracted using the QIAsymphony DSP Virus/Pathogen Kit followed by library preparation according to the veSEQ-HIV protocol~\cite{bonsall2020}. Library preparation was performed using the SMARTer Stranded Total RNA-Seq v2-PicoInputMammalian (Clontech, TakaRaBio) kit and double-stranded dual-indexed cDNA generated using in-house indexed primers. Libraries were pooled and cleaned with Agencourt AMPure XMP. Pooled libraries were hybridized to HIV-specific biotinylated 120-mer oligonucleotides (xGen Lockdown Probes, Integrated DNA Technologies) and isolated with streptavidin-conjugated beads. Captured libraries were PCR amplified prior to generation of 350-600 base pair (bp) paired-ends reads with the Illumina NovaSeq 6000 at the Oxford Genomic Centre. \par

Kraken v.0.10.5-beta~\cite{wood2014} with a custom database of human, bacterial, archael, viral, and fungal genomes was used to isolate reads of viral and unknown origin which were trimmed of adaptors and low-quality bases using trimmomatic~\cite{bolger2014} v.0.36/0.39. Trimmed reads were \textit{de novo} assembled into contigs using SPAdes~\cite{bankevich2012} and metaSPAdes~\cite{nurk2017} v.3.10. Shiver v.1.5.7~\cite{wymant2018} was used to align reads to a reference sequence constructed for each sample using these contigs. \par  

\subsection{Inference of within-host deep-sequence phylogenetic trees}
\label{sec:phsc_methods}
To improve the computational efficiency of our within-host deep-sequence phylogenetic analyses we first clustered participants with HIV into putative transmission networks as previously described (\nameref{supp_methods}) \cite{monod2024,xi2021}, and then grouped putative networks into batches for deep-sequence phylogenetic analyses. \par

Deep-sequence data belonging to participants in each batch were further processed with \textit{phyloscanner} \cite{wymant2017} v.1.8.1 to infer within-host phylogenetic trees in \var{n_all_windows} sliding windows of length \var{phsc_window_size} bp with a step size of \var{phsc_step_size} across the HIV genome as in \cite{monod2024}. As suggested in \cite{wymant2017}, this window-size was chosen to be long enough to capture sufficient within-host diversity to provide phylogenetic signal but no longer than the target read length and short enough to minimize within-window recombination. Windows spanning \textit{env} gp120 were excluded as genetic diversity in the variable loop regions~\cite{lynch2009} led to poor sequence alignment and unreliable within-host phylogenetic trees. In addition to deep-sequence data from RCCS participants, we included as phylogenetic background 113 consensus sequences from representative subtypes and circulating forms and 200 near full-length consensus sequences from Kenya, Uganda, and Tanzania (Los Alamos National Laboratory HIV Sequence Database, \href{http://www.hiv.lanl.gov}{http://www.hiv.lanl.gov}, \nameref{reference_genomes}). Within \textit{phyloscanner}, MAFFT v.7.475 \cite{katoh2002} with iterative refinement and iterative re-alignment using consistency scores was used to align sequencing reads and IQ-TREE v.2.0.3 with the GTR+F+R(Free-Rate)6 substitution model was used for phylogenetic inference~\cite{nguyen2015,minh2020}. Phylogenetic branch lengths within \textit{phyloscanner} were adjusted to account for varying substitution rates across the HIV genome as described in~\cite{ratmann2019} (\nameref{normalization_constants}). Adjusted distances can be interpreted as average distances expected in the \textit{pol} gene. The genomic coordinates of input sequence data were standardized to the coordinates of the HIV-1 HXB2 reference genome (GenBank: K03455.1). \par

For each participant, \textit{phyloscanner} was used to estimate the number of genetically distinct phylogenetic lineages (subgraphs) in each genome window using a modified parsimony algorithm. In each window, for each participant, the given phylogenetic tree was pruned to include only tips from the given participant and the specified outgroup (here, the subtype H consensus sequence). Ancestral nodes in the pruned tree were assigned to one of two states: either that of the  participant or an unsampled ``unassigned'' state (to which the outgroup and root of the phylogeny was assigned), representing the lineages that are evolutionarily ancestral to the lineages that initiated a given host's infection. To accurately assign nodes without relying on patterns of phylogenetic clustering with reference sequences, we employed a modified Sankoff minimum parsimony algorithm for ancestral state reconstruction as described in~\cite{sankoff1975,wymant2017} (in particular, see Supplementary Information 1.2 and Supplementary Figure 1 in \cite{wymant2017}). This algorithm assigns a cost ($c(n,h)$) to a state change along a lineage ending at ancestral node $n$ that is proportional to the sum of the branch lengths descendant from that node that give rise to tips form host $h$ ($l(n,h)$). As tips from all other subjects with the exception of the outgroup were pruned from the tree prior to this procedure (``single-host tree''), this is equivalent to the sum of the total branch length of the subtree with node $n$ as its root. Specifically, this cost was calculated as: 
\begin{equation}
c(n,h) = 1 + k \times l(n,h),
\end{equation}
where $k$ is a tuneable constant that controls the penalty associated with fewer host $h$ subgraphs. Traditional parsimony is recovered when $k=0$ which will always assign all tips in a single-host tree to a single subgraph, regardless of the phylogenetic branch length captured within that subgraph. As $k \to \infty$, each tip belonging to host $h$ will be assigned to a unique subgraph. Here, we parameterized $k$ with the goal of distinguishing evolution that occurred within a given host from evolution that occurred prior to HIV acquisition, in the case of multiple infection. In the case of single infection, all tips in a single-host tree will be closely related (e.g. Fig \ref{empirical_data_sum}A) and therefore we want the ancestral reconstruction that minimizes $c(n,h)$ to assign all tips to a single subgraph. In the case of a multiple infection the tips will be expected to fall into ($\ge$)2 clades with relatively small within-subgraph distances but large between-subgraph distances and we seek to parameterize $k$ such that the ancestral reconstruction minimizing $c(n,h)$ differentiates these clades into distinct subgraphs. We conservatively used a $k$ value of 15 such that $\frac{1}{k} = 0.067$, which is greater than the 99th percentile of the pairwise genetic distances between epidemiologically confirmed HIV transmission pairs \cite{ratmann2019} and comparable to within-subtype HIV genetic diversity within Rakai \cite{kim2024}.  \par

Quality filtering of inferred within-participant phylogenetic trees was performed with \textit{phyloscanner}. Specifically, within each window, subgraphs with less than three reads or less than 1\% of  reads from a particular participant were marked as putative contaminants and removed from the analysis. To mask regions with insufficient data for reliable phylogenetic inference any window with less than 30 reads from a given participant after aforementioned filter was also removed from the analysis. After filtering we identified the subgraphs with data from the deep-sequenced reads from each sequenced sample for a given participant. \par

\subsection{Bayesian model to identify multiple infections}
\label{sec:model_methods}
We developed a Bayesian statistical model to identify samples harboring multiple infections and estimate the prevalence of multiple infections in a set of deep-sequencing reads that were processed with \textit{phyloscanner}. We refer to this model as the  the deep-phylo multiple infection model (\textit{deep-phyloMI}). We first summarized the \textit{phyloscanner} output for each sample and each genomic window in terms of two binary variables, $N^{\text{obs}}_{i=1...n,w}$ (presence/absence of sequencing reads from sample $i$ in window $w$ following \textit{phyloscanner} contamination filtering) and $\MI_{i=1...n,w}$ (presence/absence of multiple subgraphs for sample $i$ in window $w$) where $n$ is the number of sequenced samples. To simplify notation below, when $N^{\text{obs}}_{i,w} = 0$ we set $\MI_{i,w} = 0$. We further summarized the data for sample $i$ into two quantities, $N^{\text{obs}}_{i} = \sum\limits_{w=1}^{n^{\text{max}}} N^{\text{obs}}_{i,w}$ and $\MI_i = \sum\limits_{w=1}^{n^{\text{max}}} \MI_{i,w}$ where ${n^{\text{max}}}$ is the number of genome windows.

\subsubsection{Base model accounting for partial sequencing success of infecting variants}
\label{sec:base_model}
We first developed a base model that accounts for partial sequencing success across the HIV genome in giving rise to the observed $N^{\text{obs}}_{i=1...n,w}$ and $\MI_{i=1...n,w}$. Working from first principles, we first derived a likelihood model for observing the pair of counts ($N^{\text{obs}}_{i}$, $\MI_i$) for the unobserved groups of samples with true multiple infection ($M_i = 1$) and single infection ($M_i = 0$), and subsequently marginalise out the unknown true multiple infection status (either $M_i = 0$ or $M_i = 1$). Among samples from multiply infected individuals ($M_i = 1$), we assumed that the probability of sequencing each of the infecting variants in window $w$ was given by $\theta_i$ for each sample $i$. The probability of sequencing at least one variant in each window is therefore $1 - (1-\theta_i)^2$ and the probability of sequencing both variants given at least one was sequenced is therefore $\frac{\theta_i}{2-\theta_i}$. Assuming sequencing success was independently and identically distributed for each sample, we obtained
\begin{subequations}
\label{eq:base-bin-ni-m1} 
    \begin{align}
    \left(N^{\text{obs}}_i|\theta_i, M_i = 1\right) & \sim \text{Binomial}^{1+}\left(n^{\text{max}}, 1-(1-\theta_i)^2\right)\\ 
    \left(\MI_i|N^{\text{obs}}_i, \theta_i, M_i = 1\right) & \sim \text{Binomial}\left(N^{\text{obs}}_i, \frac{\theta_i}{2-\theta_i}\right),
    \end{align} 
\end{subequations}
where $\text{Binomial}^{1+}$ represents the 0-truncated $\text{Binomial}$ distribution as we only consider data from individuals with \textit{phyloscanner} output in at least one genomic window, and $n^{\text{max}}$ is the total number of genomic windows. This model implicitly accounts for the presence of windows in which only a single variant was present in the \textit{phyloscanner} output due to incomplete sequencing success. For samples from individuals infected with only a single variant ($M_i = 0$), we obtained analogously 
\begin{subequations}
    \begin{align}
    \left(N_i^{\text{obs}}|\theta_i, M_i = 0\right) & \sim \text{Binomial}^{1+}\left({n^{\text{max}}}, \theta_i\right)\\ \label{eq:base-bin-ni-m0}
    \left(\MI_i|N^{\text{obs}}_i, \theta_i, M_i = 0\right) &\sim \text{Binomial}\left(N^{\text{obs}}_i, 0\right).
\end{align}
\end{subequations}
Taken together, the joint likelihood of observing the count pair  ($\MI_i$, $N^{\text{obs}}_i$) conditional on latent multiple infection status $M_i$ is given by 
\begin{equation}
P(N^{\text{obs}}_i, \MI_i | \theta_i, M_i) = P(N^{\text{obs}}_i |  \theta_i, M_i)P(\MI_i | N^{\text{obs}}_i, \theta_i, M_i).
\end{equation}
Thus, aggregating over the two unknown possible multiple infection states $M_i\in\{0,1\}$ for each sample in a finite mixture model framework, we have
\begin{equation}
\begin{split}
& P(N^{\text{obs}}_i, \MI_i | \theta_i) = \\
&\quad\quad \sum_{m = 0,1} P(N^{\text{obs}}_i | \theta_i, M_i=m)P(\MI_i | N^{\text{obs}}_i, \theta_i, M_i=m)P(M_i=m).
\end{split}
\end{equation}
One of our primary inferential targets was the individual-level probability of harboring multiple infection not conditional on observed $N^\text{obs}_i$ and $\MI_i$, which we denoted with $\delta_i = P(M_i = 1)$. Making this target explicit in the joint likelihood, we have
\begin{subequations}
    \begin{align}
    &P(N^{\text{obs}}_i, \MI_i | \theta_i, \delta_i) = \\
    &\quad\quad \delta_i \times P(N^{\text{obs}}_i | \theta_i, M_i=1)P(\MI_i | N^{\text{obs}}_i, \theta_i, M_i=1)\label{eq:lkl-MI1} \\ 
    & \quad\quad + (1-\delta_i)\times P(N^{\text{obs}}_i | \theta_i, M_i=0)P(\MI_i | N^{\text{obs}}_i, \theta_i, M_i=0),
    \end{align}
\end{subequations}
and so the log posterior distribution of the parameters $\mathbf{\theta} = (\theta_1,\dotsc,\theta_n)$, $\mathbf{\delta} = (\delta_1,\dotsc,\delta_n)$ for all the data $\mathbf{x} = \big( (N^{\text{obs}}_1, \MI_1),\dotsc,(N^{\text{obs}}_n, \MI_n)\big)$  under our model is
\begin{equation}
\log f(\mathbf{\theta}, \mathbf{\delta} | \mathbf{x}) \propto \sum_{i=1}^n \Big( \log P(N^{\text{obs}}_i, \MI_i | \theta_i, \delta_i) + \log f(\theta_i,\delta_i) \Big),
\label{eq:model_ind_posterior}
\end{equation}
where we use $f$ to denote posterior and prior densities.

\subsubsection{Base model prior densities}
In the base model, prior to observing data, we modelled the individual-level probability of multiple infection as identical for all $i$ with the prior density, 
\begin{equation}
\label{eq:model_prob_mi}
\text{logit}\left(\delta_i\right) = \delta_0 \sim \text{Normal}(0,3.16^2),
\end{equation}
with diffuse variance~\cite{stan2024}. Given the known log-linear dependency of sequencing success on log viral load~\cite{bonsall2020}, known differences in sequencing success rates by sampling protocol~\cite{monod2024}, and other factors, we specified the prior on the individual-level sequencing probability $\theta_i$ through a logistic mixed effects model. Specifically, we modeled $\text{logit}(\theta_i)$ with 
\begin{subequations}
\begin{align}
\text{logit}\left(\theta_i\right) &= 
\alpha_0 + \sigma_{\text{stz}}\alpha_1 X^{\text{amplicon}}_i + \sigma_{\text{stz}}\alpha_2 X^{\text{bait}}_i + \alpha_3 V_i+ \alpha_4 V_i^a X^{\text{amplicon}}_i + \alpha_5 V_i^bX^{\text{bait}}_i + \alpha_i \\
\alpha_0 &\sim \text{Normal}(0,2^2) \\
\alpha_3 &\sim \text{Normal}(0,2^2) \\
(\alpha_1,\alpha_2) &\sim \sigma_{\text{stz}}\times\text{stz-MVN}(0,1)\\
(\alpha_4,\alpha_5) &\sim \sigma_{\text{stz}}\times\text{stz-MVN}(0,1)\\
\alpha_i &\sim \text{Normal}(0,\sigma_{\text{ind}}^2)\\
\sigma_{\text{ind}} &\sim \text{Half-Cauchy}(0,1),
\end{align}
\label{eq:base_priors}
\end{subequations}
where $X^{\text{amplicon}}_i$ and $X^{\text{bait}}_i$ are indicator variables for whether sample $i$ was sequenced using the amplicon or bait capture approach respectively and $V_i$, $V_i^a$, and $V_i^b$ are the sample log\textsubscript{10} copies/mL values standardized to have mean zero and standard deviation 1 among all samples and among only the amplicon ($V_i^a$) and bait capture ($V_i^b$) samples, respectively. To maintain identifiability we constrain $\alpha_1 + \alpha_2 = \alpha_4 + \alpha_5 = 0$ by specifying their joint prior distributions with a zero-mean multivariate normal with a particular variance-covariance matrix described in \cite{stan2024}, such that all marginal distributions are standard normal, e.g. $\alpha_1 \sim \text{Normal}(0,1)$ and $\alpha_2 \sim \text{Normal}(0,1)$, which we represent with the notation $\text{stz-MVN}$. To maintain marginal priors with standard deviation $\sigma_{\text{stz}}=2$, we adopt a non-centered parameterisation and post-multiply the sum-to-zero random variables with $\sigma_{\text{stz}}$. Finally, $\alpha_i$ denotes an individual-level random effect.


\subsubsection{Modelling false-negative and false-positive phylogenetic observations}
\label{sec:full_model}
We extended the base model (Eq~\ref{eq:model_prob_mi}) to account for possible false-negative and false-positive phylogenetic observations, accounting for incomplete removal of false-positive observations through \textit{phyloscanner}, and/or incomplete phylogenetic identification of multiple infections due to insufficient phylogenetic background. First, among samples from individuals in which $M_i = 1$ we accounted for the scenario in which both variants are successfully sequenced in a given window but were identified as a single phylogenetic clade by \textit{phyloscanner}, i.e. false-negative observations, by modifying our data-generating model (Eq~\ref{eq:base-bin-ni-m1}) to
\begin{equation}
\left(\MI_i|N^{\text{obs}}_i, \theta_i, M_i = 1\right) \sim \text{Binomial}\left(N^{\text{obs}}_i, \frac{\theta_i}{2-\theta_i}\left(1-\lambda\right)\right),
\end{equation}
where $\lambda$ represents the false-negative rate.
We analogously accounted for the scenario in which only a single variant was sequenced but \textit{phyloscanner} spuriously assigned multiple subgraphs in a given window, i.e. false-positive observations, through a false-positive rate  $\epsilon$ in the model. We modeled false-positives among samples lacking multiple infection and among windows in multiply infected samples in which only a single variant was sequenced, which occurs with probability $2\frac{1-\theta_i}{2-\theta_i}$ when $N_{i,w} = 1$, but was spuriously assigned to two subgraphs. Note that because we did not differentiate between windows with exactly 2 and $>2$ subgraphs, we do not consider the scenario where both variants are sequenced in a true multiple infection and the two sequenced variants are spuriously assigned to 3 or 4 subgraphs. Our data generating model (Eq~\ref{eq:base-bin-ni-m0}) was updated to account for false-positives and false-negatives as:
\begin{subequations}
 \label{eq:full_model_mi_distribution}
 \begin{align}
  \left(\MI_i|N^{\text{obs}}_i, \theta_i, M_i = 1\right) &\sim \text{Binomial}\left(N^{\text{obs}}_i, \frac{\theta_i}{2-\theta_i}\left(1-\lambda\right) + 2\epsilon\frac{1-\theta_i}{2-\theta_i} \right) \\
  \left(\MI_i|N^{\text{obs}}_i, \theta_i, M_i = 0\right) &\sim \text{Binomial}\left(N^{\text{obs}}_i, \epsilon\right),
 \end{align}
\end{subequations}
with additional prior densities
\begin{subequations}
\label{eq:full_model_addl_priors}
\begin{align}
\text{logit}(\lambda) &\sim \text{Normal}(0,1)[,2.2] \\
\text{logit}(\epsilon) &\sim \text{Normal}(0,1),
\end{align}
\end{subequations}
where $[,2.2]$ represents that $\text{logit}(\lambda)$ was constrained to be $<2.2$ and all other components of the model remaining as above.


\subsubsection{Estimating risk factors of multiple infection}
\label{sec:extended_model}
We further extended the model described above (Eqs~\ref{eq:model_prob_mi}, \ref{eq:base_priors}, \ref{eq:full_model_mi_distribution}, \ref{eq:full_model_addl_priors}) to model the probability of multiple infection as dependent on potential clinical, behavioral, and/or epidemiological risk factors through a logistic regression approach. Specifically, we modeled the logit of the individual-level multiple infection prior probabilities (Eq~\ref{eq:model_prob_mi}) as a linear predictor of fixed effects,
\begin{subequations}
\label{eq:delta_i_covariates}
\begin{align}
\text{logit}\left(\delta_i\right) &= X_i^{\text{risk}}\beta = \delta_0 + \sum_{j=1}^{n_c}X_i^j \beta_j\\
\beta_j &\sim \text{Normal}(0,1), \text{ if } k_j = 1,\\
\beta_j &\sim \text{stz-MVN}(0,1), \text{ if } k_j > 1,
\end{align}
\end{subequations}
where $X^j_i$ are $1 \times k_j$ dimensional row vectors for each of $n_c$ putative multiple infection predictive covariates and $\beta_j$ are $k_j \times 1$ dimensional column vectors of fixed effect coefficients. For all categorical $j$ in $n_c$ with $k_j$ levels, we model the corresponding $k_j$ fixed effects with the sum-to-zero joint multivariate normal prior defined above to maintain identifiability.\par
%
%
We also considered a fixed effects model with Horseshoe-type shrinkage priors \cite{carvalho2009, piironen2017} on the effect sizes to handle correlated individual-level covariates. To maintain desirable sum-to-zero properties, we define a global non-negative shrinkage parameter $\tau \in [0,\infty)$, and for each categorical $j$ with $k_j$ levels $k_j$ non-negative local shrinkage parameters $\xi_{j}\in [0,\infty)^{k_j}$, and the diagonal matrix $D_j = \text{diag}(\xi_{j})$. We then specify $k_j$ sum-to-zero shrinkage effects $\beta_j$ through a joint zero-mean multivariate normal distribution with variance covariance matrix $\frac{k_j}{k_j-1}[D_j - D_j1 (1^T D_j 1 )^{-1} 1^TD_j]$, such that $0=\sum_{l=1}^{k_j} \beta_{j,l}$ and the induced marginal distributions of each $\beta_{j,l}$ are $\text{Normal}(0, \xi_{j,l}^2)$, which we refer to $\text{stz-MVN}(0, \xi_j^2)$. We incorporated the global shrinkage parameter in non-centered parameterisation through post-multiplication as in~\eqref{eq:base_priors}. Therefore, we have: 
\begin{subequations}
\label{eq:delta_i_covariates_hsp}
\begin{align}
\text{logit}\left(\delta_i\right) &= X_i^{risk}\beta = \delta_0 + \sum_{j=1}^{n_c}X_i^j \beta_j\\
\beta_j | \xi_j,\tau &\sim 
  \tau \times \text{Normal}(0,\xi_j^2), \text{ if } k_j = 1,\\
\beta_j | \xi_j,\tau &\sim   \tau \times \text{stz-MVN}(0,\xi_j^2), \text{ if } k_j > 1,\\
\xi_j &\sim  \text{Half-}t_2(0, 1)\\
\tau &\sim \text{Half-Cauchy}(0,1),
\end{align}
\end{subequations}
where we modelled the $\xi_j$ with t-distributions with 2 degrees of freedom instead of Cauchy distributions to ease numerical sampling. \par
As above, the number of lifetime sex partners included missing and ambiguous responses (e.g. ``3+''), and these values were estimated as additional random variables in the Bayesian inference, assuming they were missing at random within sex, age, and community type, using lognormal prior distributions specific to these strata defined by the non-missing responses as above. Imputed values for missing responses were limited to the range [1,60] and responses of ``3+'' were limited to the range [3,60].  
\par

\subsubsection{Parameter estimation}
We estimated joint posterior distributions numerically using Hamiltonian Monte Carlo \cite{betancourt2013} with the No-U-Turn Sampler \cite{hoffman2014} implemented in Stan \cite{stan2024} and accessed through cmdStanR v.2.36.0 \cite{gabry2023} in R. For all analyses, four independent chains with 2,000 iterations of warm up and 2,000 iterations of sampling were run. A target acceptance rate of 0.8 was used for all analyses with the exception of those that employed shrinkage priors where a target acceptance rate of 0.95 was used to avoid divergent transitions. Convergence was assessed using the $\hat{R}$ statistic, bulk and tail effective sample sizes (ESS) for each parameter \cite{vehtari2021}, and visual inspection of trace and pairs plots (\nameref{model_diagnostics}). \par

\subsubsection{Generated quantities}
\setcounter{secnumdepth}{4}
Based on the estimated parameter distributions of the models described above, we generated a number of quantities to aid in interpretation of our results. 

\paragraph{Posterior probabilities of individual-level multiple infection.}
We computed the posterior probabilities of individual-level multiple infection directly from Monte Carlo samples of the joint posterior density via
\begin{equation}
\label{eq:ind_prob_mi}
\begin{split}
P(M_i = 1 | N^{\text{obs}}_i, \MI_i, {n^{\text{max}}}) = \\
\int P(M_i = 1 | N^{\text{obs}}_i, \MI_i, {n^{\text{max}}}, \theta_i, \delta_i, \lambda, \epsilon)P(\theta_i, \delta_i, \lambda | N^{\text{obs}}_i, \MI_i)d (\theta_i,\delta_i,\lambda),
\end{split}
\end{equation}
by taking for each individual $i$ all Monte Carlo samples of the posterior density of $(\theta_i, \delta_i, \lambda)$, evaluating $P(M_i = 1 | N^{\text{obs}}_i, \MI_i, {n^{\text{max}}}, \theta_i, \delta_i, \lambda, \epsilon)$ according to:
\begin{equation}
P(M_i = 1 | N^{\text{obs}}_i, \MI_i, {n^{\text{max}}}, \theta_i, \delta_i, \lambda, \epsilon) = \frac{\delta_i \times P(N^{\text{obs}}_i, \MI_i | {n^{\text{max}}}, \theta_i, \lambda, \epsilon, M_i = 1)}{P(N^{\text{obs}}_i, \MI_i, | {n^{\text{max}}}, \theta_i, \delta_i,  \lambda, \epsilon)}. 
\end{equation}
and calculating the expectation across these. 


\paragraph{Prevalence of multiple infection in the study sample.} 
Following from prior work on Bayesian latent class models with covariates \cite{thomson1998, shi2005, proustlima2007, williford2020, samerei2021, sotresalvarez2010}, under the base model (Eq~\ref{eq:model_prob_mi}) the posterior estimate of the prevalence of multiple infection in the study sample is given by: 
\begin{equation}
    \bar{\delta} = \text{inverse-logit}\left(\delta_0\right) = \frac{\exp(\delta_0)}{1+\exp(\delta_0},
\end{equation}
where $\delta_0$ is from the joint posterior density of the model defined by Eqs~\ref{eq:model_prob_mi}, \ref{eq:base_priors}, \ref{eq:full_model_mi_distribution}, and \ref{eq:full_model_addl_priors}.
In the presence of modeled risk factors (Eqs~\ref{eq:delta_i_covariates} and \ref{eq:delta_i_covariates_hsp}),
the prevalence of multiple infections in the study sample will vary based on sub-groups $s$ defined by $X^{risk}$. In the case where $X^\text{risk}$ contains only the covariates used to define $s$:
\begin{equation}
        \label{eq:strata_prob_mi}
    \bar{\delta_{s}} = \text{inverse-logit} \left( \delta_0 + X^\text{risk}_{s} \beta \right). 
\end{equation}
Finally, we estimated the prevalence in a target population (e.g. the entire sample of sequenced viremic RCCS participants) through post-stratification: 
\begin{equation}
    \bar{\delta} = \frac{\sum\limits_{s=1}^{S}Q_s\bar{\delta_{s}}}{\sum\limits_{s=1}^S Q_s},
\end{equation}
where $Q_s$ are the number of sampled individuals in each of the $S$ sub-populations $s$ and $\bar{\delta_{s}}$ are the sub-group specific prevalence estimates from Eq~\ref{eq:strata_prob_mi}. 

\paragraph{Prevalence and risk ratios of harboring multiple infection associated with epidemiological covariates.} 
We calculated a posterior estimate for the prevalence risk ratio (PRR) of multiple infections in 
epidemiological strata $s*$ as compared to strata $s$ as
\begin{equation}
    \text{PRR}_{s^*,s} = \frac{\bar{\delta}_{s^*}}{\bar{\delta}_{s}}.
    \label{eq:strata_prev_ratio}
\end{equation}
In the case where $X^{\text{risk}}$ contained additional covariates beyond those used to define $s^*$ from $s$ we estimated a multivariate risk ratio (RR) associate with the covariate(s) that distinguish $S^*$ from $S$ by calculating the ratio of the estimated risk of multiple infection for each $i$ 
were they to be in $s^*$ and $s$, while holding all other covariates at their observed values (based on the design matrices $X^\text{risk}_{i | i \in s^*}$ and $X^\text{risk}_{i | i \in s}$, respectively): 
\begin{equation}
    \text{RR}_{s^*,s} = \frac{1}{n}\sum\limits_{i=1}^n {\frac{\text{inverse-logit} \left( \delta_0 + X^\text{risk}_{i | i \in s^*} \beta \right )}{\text{inverse-logit} \left( \delta_0 + X^\text{risk}_{i | i \in s} \beta \right )}}.
\end{equation}.


\paragraph{Post-stratification adjustments.}
Finally, because sequence data was not available for all viremic participants with HIV in our study population, we employed post-stratification based on prevalence estimates in epidemiological sub-groups $s$ (Eq~\ref{eq:strata_prob_mi}) to estimate the prevalence of multiple infections in the population under study (viremic study participants) \cite{Little1993}. Specifically, we calculated
\begin{equation}
    \bar{\delta^*} = \frac{\sum\limits_{s=1}^{j}W_s\bar{\delta_{s}}}{\sum\limits_{s=1}^j W_s},
\end{equation}
where $W_s$ is the estimated population size or estimated relative population size of sub-group $s$. The population prevalence ratio between two non-overlapping composite sub-groups can therefore be calculated as in Eq~\ref{eq:strata_prev_ratio}. We performed post-stratification based on the total number of participant-visits from viremic PLHIV stratified by age ((14, 24], (24, 34], and (34, 49] years), sex, and community type. Because viral load measurements were not routinely conducted for all PLHIV in the 2010 and 2012 survey rounds we calculated population-sizes using only participant-visits in the 2014-2019 survey rounds. 

\subsection{Simulation study}
We used simulations to validate our inference model. For all simulations, we simulated data for ${n^{\text{max}}} = 29$ genome windows in $n = 2,000$ samples which were assigned a normalized log\textsubscript{10} viral load ($V_i$) with random draws from a $N(0,1)$ distribution. For all samples, $\alpha_i$ was drawn from a $N(0,1)$ distribution and $\theta_i$ calculated as $\alpha_0 + \alpha_1 V_i + \alpha_i$ with $\alpha_0 = 2$ and $\alpha_1 = 2$. Under these parameters, we generated three simulated data sets as described below.

\subsubsection{Base simulation}
\label{sec:base_simulation}
\begin{subequations}
\begin{align}
M_i &= [{1}_{\times 100}] \oplus [{0}_{\times 1900}]\\
(N^{\text{obs}}_i | M_i = 0) &\sim \text{Binomial}^{1+}(29, \phi_i) \\
(N^{\text{obs}}_i | M_i = 1) &\sim \text{Binomial}^{1+}(29, (1-(1-\phi_i)^2) \\
(\MI_i | M_i = 0)  &\sim \text{Binom}\left(N^{\text{obs}}_i, 0\right)\\
(\MI_i | M_i = 1) &\sim \text{Binomial}\left( N^{\text{obs}}_i, \frac{\phi_i}{2-\phi_i}\right),
\end{align}
\end{subequations}
where $[{x}_{\times n}]$ represents a vector of $x$ repeated $n$ times and $\oplus$ represents concatenation of two vectors. \par

\subsubsection{Full simulation}
\label{sec:full_simulation}
\begin{subequations}
\begin{align}
M_i &= [{1}_{\times 100}] \oplus [{0}_{\times 2000}]\\
(N^{\text{obs}}_i | M_i = 0) &\sim \text{Binomial}^{1+}(29, \phi_i) \\
(N^{\text{obs}}_i | M_i = 1) &\sim \text{Binomial}^{1+}(29, (1-(1-\phi_i)^2) \\
(\MI_i | M_i = 0)  &\sim \text{Binom}\left(N^{\text{obs}}_i, \epsilon\right)\\
(\MI_i | M_i = 1) &\sim \text{Binom}\left(N^{\text{obs}}_i, \frac{\phi_i}{2-\phi_i}(1-\lambda) + 2\epsilon\frac{1-\phi_i}{2-\phi_i}\right) \\
\lambda &= 0.30 \\
\epsilon &= 0.01.
\end{align}
\end{subequations}
Additional simulations from this simulation model were generated with all other parameters held constant except (A): $\sum_i^n M_i = 0, 300, 600$, (B): $\lambda = 0.10, 0.20, 0.40$, and (C): $\epsilon = 0, 0.005, 0.05$.

\subsubsection{Extended simulation}
\label{sec:extended_simulation}
\begin{subequations}
\begin{align}
M_i &= [{1}_{\times 150}] \oplus [{0}_{\times 1850}]\\
X^{\text{risk}}_{,1} &= [{1}_{\times 100}] \oplus [{0}_{\times 50}] \oplus [{1}_{\times 900}] \oplus [{0}_{\times 950}]\\
X^{\text{risk}}_{i,2-5} &\sim  \text{shuffle}([{1}_{\times 1000}] \oplus [{0}_{\times 1000}]) \\
(N^{\text{obs}}_i | M_i = 0) &\sim \text{Binomial}^{1+}\left(29, \phi_i\right) \\
(N^{\text{obs}}_i | M_i = 1) &\sim \text{Binomial}^{1+}\left(29, (1-(1-\phi_i)^2\right) \\
(\MI_i | M_i = 0) &\sim \text{Binomial}\left(N^{\text{obs}}_i, \epsilon\right) \\
(\MI_i | M_i = 1) &\sim \text{Binomial}\left(N^{\text{obs}}_i, \frac{\phi_i}{2-\phi_i}(1-\lambda) + 2\epsilon\frac{1-\phi_i}{2-\phi_i}\right),
\end{align}
\end{subequations}
where $X^{\text{risk}}_{i,j}$ represents the entry in the \textit{ith} row and \textit{jth} column of the design matrix $X^{\text{risk}}$ and shuffle($v$) denotes shuffling the elements of $v$. \par

\subsection{Data analysis and visualization}
All data analysis was conducted in R v.4.4.1 \cite{R} using the tidyverse~\cite{wickham2019} with dplyr v.1.1.4~\cite{wickham2023d}, tibble v.3.2.1~\cite{muller2023}, and tidyr v.1.3.1~\cite{wickham2024}. Haven v.2.5.4~\cite{wickham2023h} was used to parse a subset of input data files. Visualization of data and results was done using ggplot2 v.3.5.1~\cite{wickham2016} with bayesplot v.1.11.1~\cite{gabry2024,gabry2019}, cowplot v.1.1.3~\cite{wilke2024}, and patchwork v.1.2.0.~\cite{pedersen2024}. Phylogenetic trees were manipulated and visualized using ape v.5.8~\cite{paradis2019}, ggtree v.3.12.0~\cite{yu2022, xu2022, yu2020, yu2018, yu2017}, phytools v.2.1.-1~\cite{revell2024}, and tidytree v.0.4.6~\cite{yu2022}. Highest posterior density intervals were calculated with HDInterval v.0.2.4~\cite{meredith2022} and convergence statistics were assessed with posterior v.1.6.0~\cite{posterior}. Preliminary analyses and model fitting was performed using fitdistrplus v.1.1-11~\cite{delignette-muller2015}. \par

\subsection{Data and code availability}
 Processed \textit{phyloscanner} output, de-identified epidemiological metadata, and all analysis and visualization code is available at \href{https://github.com/m-a-martin/rccs_hiv_moi}{https://github.com/m-a-martin/rccs\_hiv\_moi}. 
% will change github links to zotero upon acceptance
HIV consensus sequences are available from Zenodo (\href{https://doi.org/10.5281/zenodo.10075814}{https://doi.org/10.5281/zenodo.10075814}) and the PANGEA-HIV sequence repository (\href{https://github.com/PANGEA-HIV/PANGEA-Sequences}{https://github.com/PANGEA-HIV/PANGEA-Sequences}) as open-access dataset under the CC-BY-4.0 license \cite{pangea}. \par
HIV-1 deep-sequence reads can be requested from PANGEA-HIV under a managed access policy due to privacy and ethical reasons, which aligns with UNAIDS ethical guidelines. The process for accessing data, the PANGEA-HIV data sharing policy and a detailed description of what data are available is described at \href{https://www.pangea-hiv.org/join-us}{https://www.pangea-hiv.org/join-us}. For more information contact PANGEA project manager Lucie Abeler-Dörner (\href{mailto:lucie.abeler-dorner@ndm.ox.ac.uk}{lucie.abeler-dorner@ndm.ox.ac.uk}). The time frame for a response to requests is 2–4 weeks. \par
Additional cohort data can be requested from RHSP. Because of privacy and ethical reasons, RHSP maintains a controlled access data policy for corresponding epidemiological metadata and corresponding data collection tools. In brief, RHSP policy requires individuals to submit an RHSP data request form (available upon request from \href{mailto:info@rhsp.org}{info@rhsp.org} or \href{mailto:gkigozi@rhsp.org}{gkigozi@rhsp.org}) and a brief concept note (one or two pages) detailing their research questions and methods. In addition, researchers are asked to provide a curriculum vitae/resume along with proof of human subjects research training. The time frame for a response to requests is 2–4 weeks.
% will change github links to zotero upon acceptance
\par

%----------------------------------------------------------------------------------------
%   RESULTS
%----------------------------------------------------------------------------------------
\section{Results}
%\subsection{Study population}
\subsection{Phylogenetic signatures of multiple infection in population-based pathogen surveillance}
Between \var{min_survey_year} and \var{max_survey_year}, \var{n_participants} participants contributed to the RCCS in \var{n_participant_visits} visits over \var{n_rounds_char} survey rounds. Overall, \var{n_hiv_participants} participants were HIV seropositive and \var{n_viremic_participants} were viremic (plasma viral load $\ge$ 1,000 copies/mL) at one of their visits (\nameref{epi_table} and \nameref{missing_data_table}). Of these, \var{empirical_age_sex_comm_n_participant_visits} individuals were sampled between \var{min_survey_date} and \var{max_survey_date}, had HIV RNA deep-sequence data available at minimum quality criteria for deep-sequence phylogenetic analysis, and were identified as a member of a putative transmission network (Table~\ref{seq_table}, \nameref{poststrat_table}, and \nameref{supp_methods}). Availability of sequence data among viremic participants was generally higher
among men, from residents of fishing communities, and from participants aged 25-34 years. \par

We next inferred within-host phylogenies from deep-sequencing reads in twenty-nine \var{phsc_window_size} bp non-overlapping genomic windows using \textit{phyloscanner} (\nameref{genome_window_sensitivity}), which captured evolutionary relationships of HIV variants within individual participants. Sequencing coverage varied significantly between samples (median [interquartile range (IQR)]: \var{median_unique_N_obs_bp} [\var{iqr_unique_N_obs_bp}] bp, \nameref{vl_seq_success}A) but was generally higher among bait capture sequenced samples and samples with higher viral load. Across the genome, sequencing success was highest in \textit{gag} (Fig~\ref{empirical_data_sum}F, \nameref{vl_seq_success}B), likely due to differential amplification efficiency of the primers used in the amplicon sequencing approach \cite{ratmann2017}. 

% -- TABLE 1 --%
\begin{table}[!ht]
%TBL_ADJUST_WIDTH_START 
\centering
\caption{
{\bf Characteristics of the study sample obtained from population-level HIV deep-sequence surveillance in the Rakai Community Cohort Study, \protect\var{min_survey_year}-\protect\var{max_survey_year}, stratified by availability of deep-sequence data.}}
\begin{tabular}[t]{|l|c|c|c|c|}
\hline
&\multicolumn{4}{c|}{Participants living with HIV} \\
& & & \multicolumn{2}{c|}{Processed w/ PHSC} \\
& All & Viremic & $n$ (\%) & \% of viremic (95\% CI) \\ \thickhline

Overall & \var{n_hiv_participants} & \var{n_viremic_participants} & \var{n_unique_participants_N_obs_gt_0} & \\ \hline
Survey Round & & & & \\ \hline
\hspace{3mm}\var{round_14_median_year} & 
\var{n_hiv_participant_round_14} (\var{p_hiv_participant_round_14}\%) & \var{n_viremic_participant_round_14} (\var{p_viremic_participant_round_14}\%) & 
  \var{n_seq_viremic_participant_round_14} (\var{p_seq_viremic_participant_round_14}\%) & \var{p_sample_seq_participant_round_14} \\ \hline
\hspace{3mm}\var{round_15_median_year} & 
\var{n_hiv_participant_round_15} (\var{p_hiv_participant_round_15}\%) & \var{n_viremic_participant_round_15} (\var{p_viremic_participant_round_15}\%) & 
  \var{n_seq_viremic_participant_round_15} (\var{p_seq_viremic_participant_round_15}\%) & \var{p_sample_seq_participant_round_15}\\ \hline
\hspace{3mm}\var{round_16_median_year} & \var{n_hiv_participant_round_16} (\var{p_hiv_participant_round_16}\%) & \var{n_viremic_participant_round_16} (\var{p_viremic_participant_round_16}\%) & 
  \var{n_seq_viremic_participant_round_16} (\var{p_seq_viremic_participant_round_16}\%) & \var{p_sample_seq_participant_round_16}\\ \hline
\hspace{3mm}\var{round_17_median_year} & \var{n_hiv_participant_round_17} (\var{p_hiv_participant_round_17}\%) & \var{n_viremic_participant_round_17} (\var{p_viremic_participant_round_17}\%) & 
  \var{n_seq_viremic_participant_round_17} (\var{p_seq_viremic_participant_round_17}\%) & \var{p_sample_seq_participant_round_17}\\ \hline
\hspace{3mm}\var{round_18_median_year} & \var{n_hiv_participant_round_18} (\var{p_hiv_participant_round_18}\%) & \var{n_viremic_participant_round_18} (\var{p_viremic_participant_round_18}\%) & 
  \var{n_seq_viremic_participant_round_18} (\var{p_seq_viremic_participant_round_18}\%) & \var{p_sample_seq_participant_round_18}\\ \hline
\hspace{3mm}\var{round_19_median_year} & \var{n_hiv_participant_round_19} (\var{p_hiv_participant_round_19}\%) & \var{n_viremic_participant_round_19} (\var{p_viremic_participant_round_19}\%) & 
  \var{n_seq_viremic_participant_round_19} (\var{p_seq_viremic_participant_round_19}\%) & \var{p_sample_seq_participant_round_19}\\ \hline

Sex & & & &  \\ \hline
\hspace{3mm}Female & \var{n_hiv_participant_sex_F} (\var{p_hiv_participant_sex_F}\%) & \var{n_viremic_participant_sex_F} (\var{p_viremic_participant_sex_F}\%) & 
  \var{n_seq_viremic_participant_sex_F} (\var{p_seq_viremic_participant_sex_F}\%) & \var{p_sample_seq_participant_sex_F} \\ \hline
\hspace{3mm}Male & \var{n_hiv_participant_sex_M} (\var{p_hiv_participant_sex_M}\%) &  \var{n_viremic_participant_sex_M} (\var{p_viremic_participant_sex_M}\%) & 
  \var{n_seq_viremic_participant_sex_M} (\var{p_seq_viremic_participant_sex_M}\%) & \var{p_sample_seq_participant_sex_M} \\ \hline

Commiunity type & & & & \\ \hline
\hspace{3mm}Inland & \var{n_hiv_participant_comm_type_inland} (\var{p_hiv_participant_comm_type_inland}\%) & \var{n_viremic_participant_comm_type_inland} (\var{p_viremic_participant_comm_type_inland}\%) & 
  \var{n_seq_viremic_participant_comm_type_inland} (\var{p_seq_viremic_participant_comm_type_inland}\%) & \var{p_sample_seq_participant_comm_type_inland} \\ \hline
\hspace{3mm}Fishing & \var{n_hiv_participant_comm_type_fishing}(\var{p_hiv_participant_comm_type_fishing}\%) & \var{n_viremic_participant_comm_type_fishing}(\var{p_viremic_participant_comm_type_fishing}\%) &
  \var{n_seq_viremic_participant_comm_type_fishing} (\var{p_seq_viremic_participant_comm_type_fishing}\%) & \var{p_sample_seq_participant_comm_type_fishing}\\ \hline

Age & & & & \\ \hline
\hspace{3mm}(14, 24] & \var{n_hiv_participant_age_cat_coarse_(14,24]} (\var{p_hiv_participant_age_cat_coarse_(14,24]}\%) & \var{n_viremic_participant_age_cat_coarse_(14,24]} (\var{p_viremic_participant_age_cat_coarse_(14,24]}\%) &
  \var{n_seq_viremic_participant_age_cat_coarse_(14,24]} (\var{p_seq_viremic_participant_age_cat_coarse_(14,24]}\%) & \var{p_sample_seq_participant_age_cat_coarse_(14,24]} \\ \hline
\hspace{3mm}(24, 34] & \var{n_hiv_participant_age_cat_coarse_(14,24]} (\var{p_hiv_participant_age_cat_coarse_(24,34]}\%) & \var{n_viremic_participant_age_cat_coarse_(14,24]} (\var{p_viremic_participant_age_cat_coarse_(24,34]}\%) &
  \var{n_seq_viremic_participant_age_cat_coarse_(24,34]} (\var{p_seq_viremic_participant_age_cat_coarse_(24,34]}\%) & \var{p_sample_seq_participant_age_cat_coarse_(24,34]} \\ \hline
\hspace{3mm}(34, 49] & \var{n_hiv_participant_age_cat_coarse_(34,49]} (\var{p_hiv_participant_age_cat_coarse_(34,49]}\%) & \var{n_viremic_participant_age_cat_coarse_(34,49]} (\var{p_viremic_participant_age_cat_coarse_(34,49]}\%) &
  \var{n_seq_viremic_participant_age_cat_coarse_(34,49]} (\var{p_seq_viremic_participant_age_cat_coarse_(34,49]}\%) & \var{p_sample_seq_participant_age_cat_coarse_(34,49]} \\ \hline

Viral load (log\textsubscript{10} copies/mL) & & & & \\ \hline
\hspace{3mm}(3, 3.5] & & \var{n_viremic_participant_log10_copies_cat_(3,3.5]} (\var{p_viremic_participant_log10_copies_cat_(3,3.5]}\%) & 
  \var{n_seq_viremic_participant_log10_copies_cat_(3,3.5]} (\var{p_seq_viremic_participant_log10_copies_cat_(3,3.5]}\%) & \var{p_sample_seq_participant_log10_copies_cat_(3,3.5]} \\ \hline
\hspace{3mm}(3.5, 4] & & \var{n_viremic_participant_log10_copies_cat_(3.5,4]} (\var{p_viremic_participant_log10_copies_cat_(3.5,4]}\%) & 
  \var{n_seq_viremic_participant_log10_copies_cat_(3.5,4]} (\var{p_seq_viremic_participant_log10_copies_cat_(3.5,4]}\%) & \var{p_sample_seq_participant_log10_copies_cat_(3.5,4]}\\ \hline
\hspace{3mm}(4, 4.5] & & \var{n_viremic_participant_log10_copies_cat_(4,4.5]} (\var{p_viremic_participant_log10_copies_cat_(4,4.5]}\%) & 
  \var{n_seq_viremic_participant_log10_copies_cat_(4,4.5]} (\var{p_seq_viremic_participant_log10_copies_cat_(4,4.5]}\%) & \var{p_sample_seq_participant_log10_copies_cat_(4,4.5]}\\ \hline
\hspace{3mm}(4.5, 5] & & \var{n_viremic_participant_log10_copies_cat_(4.5,5]} (\var{p_viremic_participant_log10_copies_cat_(4.5,5]}\%) & 
  \var{n_seq_viremic_participant_log10_copies_cat_(4.5,5]} (\var{p_seq_viremic_participant_log10_copies_cat_(4.5,5]}\%) & \var{p_sample_seq_participant_log10_copies_cat_(4.5,5]}\\ \hline
\hspace{3mm}(5, $\infty$] & & \var{n_viremic_participant_log10_copies_cat_(5,Inf]} (\var{p_viremic_participant_log10_copies_cat_(5,Inf]}\%) & 
  \var{n_seq_viremic_participant_log10_copies_cat_(5,Inf]} (\var{p_seq_viremic_participant_log10_copies_cat_(5,Inf]}\%) & \var{p_sample_seq_participant_log10_copies_cat_(5,Inf]}\\ \hline
\end{tabular}
\begin{flushleft} \begin{small}For each participant, includes data from the participant-visit processed with PHSC if applicable or the participant-visit with the highest viral load, using the first visit in the case of ties or for people not living with HIV. Viremic participants excludes individuals living with HIV with suppressed viral load or missing viral load data. Viral load testing was not routinely conducted in earlier study rounds and was available for \protect \var{p_r14_r15_nonmissing_vl}\%  of participant-visits contributed by people living with HIV in the \protect \var{round_14_median_year} and \protect \var{round_15_median_year} rounds. In recent rounds, viral load testing is routinely conducted and is available for \protect \var{p_r16_r19_nonmissing_vl}\% of participant-visits contributed by people living with HIV in the \protect \var{round_16_median_year}- \protect \var{round_19_median_year} surveys. Percentages represent the row percentages within each category. 
Binomial confidence intervals were calculated using the Agresti–Coull method. PHSC = phyloscanner. \end{small} \end{flushleft}

\label{seq_table}
%TBL_ADJUST_WIDTH_END
\end{table}

% -- FIGURE 1 --%
\begin{figure}[!ht]
\ifthenelse{\boolean{includefigs}}{
\centering
\includegraphics[width=0.6\textwidth]{../figures/empirical_data_sum.pdf}
}{}
\caption{{\bf Empiric phylogenetic multiple infection signatures from \protect \var{n_unique_N_obs_gt_0} samples from people with viremic HIV in the Rakai Community Cohort Study, \protect \var{min_survey_year}-\protect\var{max_survey_year}.}
(A) Representative within-host phylogenetic tree lacking evidence of multiple phylogenetic subgraphs. (B) Representative within-host phylogenetic tree with two subgraphs as indicated by the green and blue shading of the tips. (C) Distribution of branch length distance between the MRCAs of the two subgraphs with the most sequencing reads in all genome windows windows with $\ge2$ subgraphs from all samples. Bins are shaded according to the 95th and 50th percentile. Vertical dotted line indicates median value. Binwidth is calculated such that there are approximately 50 bins across the range of observed values. (D) Per-sample number of non-overlapping genome windows with sequence data versus the number of non-overlapping genome windows with multiple subgraphs. Samples with at least one window with multiple subgraphs are shown in purple. Points have been jittered along both the X and Y axes for visual clarity. Dotted line shows modeled prediction in the absence of false-positive or false-negative multiple subgraph windows. Marginal densities are shown at right and above the scatter-plot. (E) Schematic of the HIV genome based on the coordinates from HXB2 (Genbank: K03455.1). (F) Number of samples with sequence data in each of the 29 non-overlapping genome windows. (G) Number of samples with evidence of multiple subgraphs in each of the \protect \var{n_unique_windows} non-overlapping genome-windows.}
\label{empirical_data_sum}
\end{figure}


%\subsection{Inference of within-host phylogenetic trees across the HIV genome}
To characterise phylogenetic signatures of multiple infection, we used \textit{phyloscanner} to identify distinct co-circulating variants among participants with viremic HIV (\nameref{Materials and methods} and Fig~\ref{empirical_data_sum}A,B).
We tabulated the number ($\MI_i$) of genome windows in which distinct phylogenetic lineages (phylogenetic subgraphs) were observed. The median genetic distance between the most recent common ancestors of subgraphs in genome windows with multiple subgraphs was \var{median_multi_subgraph_dist} [IQR: \var{iqr_multi_subgraph_dist}] substitutions/site (Fig~\ref{empirical_data_sum}C, \nameref{empirical_cophenetic_dists}), which is consistent with contemporary circulating genetic diversity within Rakai \cite{kim2024, ratmann2019}. 
%
Empirically, \var{n_unique_MI_obs_gt_0}   (\var{p_unique_MI_obs_gt_0}\%) samples had multiple subgraphs in at least one of the \var{n_unique_windows} non-overlapping windows (Fig~\ref{empirical_data_sum}D).  Among these, the proportion of sequenced windows in which multiple subgraphs were observed varied considerably, but was generally relatively rare (median [IQR] \var{p_unique_N_obs_w_MI_obs_median} [\var{p_unique_N_obs_w_MI_obs_iqr}]\% of sequenced windows for each sample, \var{n_unique_MI_obs_median} [\var{n_unique_MI_obs_iqr}] windows total). We observed a clear dependence of the ability to identify multiple subgraphs on sequencing success as quantified by genome coverage in the \textit{phyloscanner} output. Of those samples with sequence data in all genome windows, \var{p_unique_MI_obs_gt_0_N_obs_29}\% (\var{n_unique_MI_obs_gt_0_N_obs_29}/\var{n_unique_N_obs_29}) had at least one window with multiple subgraphs compared to \var{p_unique_MI_obs_gt_0_N_obs_lt_29}\% (\var{n_unique_MI_obs_gt_0_N_obs_lt_29} / \var{n_unique_N_obs_lt_29}) among the remaining samples. Multiple subgraph windows were more common in the genome windows corresponding to \textit{gag}, \textit{env}, and \textit{nef}, likely reflecting circulating genetic diversity in these regions with higher substitution rates \cite{patino2017}. Previous studies of HIV multiple infection in this setting have used amplicon-based deep-sequencing of two regions in p24 (1427 - 1816) and gp41 (7941 - 8264) regions \cite{redd2011, redd2012, redd2014}. Of \var{n_unique_N_obs_gt_0_amplicon} sequenced participants with data in windows spanning these regions (\nameref{genome_window_sensitivity}), \var{n_unique_MI_obs_gt_0_amplicon} (\var{p_unique_MI_obs_gt_0_amplicon}\%) had multiple subgraphs in one of the regions. \par 

\subsection{Bayesian model to identify multiple infections from pathogen deep-sequence data}
The observed dependence between phylogenetically identified samples with multiple infection and successful genome sequencing implies it is difficult to deduce the underlying prevalence of multiple infections from the empirical data without a statistical model that accounts for partial sequencing success, false-positive multiple subgraphs, and false-negative unique subgraphs within hosts. Specifically, because identification of multiple infection requires successful sequencing of both variants and genetic divergence between those variants, there is inherently more uncertainty in multiple infection status when sequencing success is poor or when infecting variants are genetically related in the sequenced region of the genome. Further, contamination or sequencing errors may give rise to spurious within-host genetic diversity and thereby inflate the estimated prevalence of multiple infection. \par

Therefore, we constructed a Bayesian model accounting for partial sequencing success to estimate the probabilities that each individual harbors a multiple infection, prevalence of multiple infection among deep-sequenced viremic participants, and  risk factors for multiple infection (\nameref{Materials and methods}). We first verified that we were able to accurately estimate model parameters on simulated test data  in the presence of incomplete sequencing success (\nameref{base_simulation_base_table}). Next, we investigated the impact of false-positive and false-negative observations, as empiric analyses of RCCS deep-sequence data indicated that false-negative rates were likely substantial in that among samples with $\MI_i > 0$, the observed $\MI_i$ values for a given number of sequenced windows ($N^{\text{obs}}_i$) was less than expected based on our model (\nameref{supp_methods} and  Fig~\ref{empirical_data_sum}D). We found that failing to account for these errors led to an overestimation of the prevalence of multiple infections on simulated data (Fig~\ref{full_simulation_full_fit}A and \nameref{full_simulation_base_table}). This prompted us to explicitly include false-positive and false-negative detection rates in our model as free parameters (Eq~\ref{eq:full_model_mi_distribution}). With this, we found that model parameters could be accurately estimated on simulated data (Fig~\ref{full_simulation_full_fit}B-H and \nameref{full_simulation_full_table}). Model performance was robust across simulations covering a range of reasonable values of the prevalence of multiple infections as well as false-positive and false-negative rates of multiple subgraph observation(\nameref{full_simulation_sensitivity_delta}, \nameref{full_simulation_sensitivity_lambda}, and \nameref{full_simulation_sensitivity_epsilon}). \par 

% -- FIGURE 2 --%
\begin{figure}[!ht]
\ifthenelse{\boolean{includefigs}}{
\centering
\includegraphics[width=0.75\textwidth]{../figures/full_simulation_full_fit.pdf}
}{}
\caption{{\bf Verification of model accuracy for estimating multiple infection prevalence on simulated data with incomplete sequencing success and false-negative and false-positive observations.}
(A) Number of windows with sequence data (x-axis) v. number of windows with multiple subgraphs (y-axis) for each simulated sample. Data from multiply infected samples is highlighted in red. Marginal distributions are shown at right and above. (B) Estimated posterior probability of multiple infection for each sample. Confidence bounds represent the 95\% highest posterior density. Data for each sample is shaded as in (A). (C-H) Posterior distributions of the baseline sequencing success ($\alpha_0$, C), dependence of sequencing success on viral load (log$_{10}$ copies/mL) standardized to mean = 0 and standard deviation = 1.  ($\alpha_1$, D), standard deviation of per-individual sequencing success random effect ($\sigma_{ind}$, E),
the multiple subgraph false-negative rate ($\lambda$, F), the multiple subgraph false-positive rate ($\epsilon$, G), and
the population prevalence of multiple infections ($\bar{\delta}$, H). Posterior distributions in (C-H) bins are shaded according to the 95\% and 50\% HPD. Histogram bin width is calculated such that there are approximately 50 bins over the range of the plotted values. True values are shown as vertical dotted lines.}
\label{full_simulation_full_fit}
\end{figure}

To identify risk factors for multiple infection among people living with HIV,  we formulated an extended model in which individual-level prior multiple infection probabilities are described with a logit linear predictor of putative risk factors (Eq~\ref{eq:delta_i_covariates}). On simulated data, this model accurately estimated the true risk ratio associated with a covariate leading to a two-fold higher probability of harboring a multiple infection (risk ratio (RR) median [95\% HPD] \var{ext_ext_fit_multivar_rr_mi_risk_factor_MI_1=='1':risk_factor_MI_1=='0'_median} [\var{ext_ext_fit_multivar_rr_mi_risk_factor_MI_1=='1':risk_factor_MI_1=='0'_lower} - \var{ext_ext_fit_multivar_rr_mi_risk_factor_MI_1=='1':risk_factor_MI_1=='0'_upper}]) in the context of four additional background null covariates (\nameref{ext_simulation_ext_table}). \par


 \subsection{Prevalence of HIV multiple infections among sequenced participants}
 We next considered estimating the prevalence of multiple infection in the sequenced sample of \var{n_unique_participants_N_obs_gt_0} participants living with viremic HIV. In a model accounting for partial sequencing success and false-positive and false-negative observations of multiple subgraphs we estimate that \var{empirical_age_sex_comm_fit_ind_prob_mi_ge_50_median} (\var{empirical_age_sex_comm_fit_ind_prob_mi_ge_50_median_percent}\%) of the sequenced viremic PLHIV had a median posterior probability of multiple infection greater than 50\% when allowing the probability of multiple infection ($\delta_i$) to vary by age, sex, and community type (Fig~\ref{empirical_age_sex_comm_fit}A,B, \nameref{empirical_age_sex_comm_log_prob_mi}). Our empirical analyses above demonstrated that the number of genome windows with multiple subgraphs is less than would be expected in the absence of false-negatives (Fig~\ref{empirical_data_sum}D). In line with this observation, the model estimated a high false-negative rate (median [95\% HPD] \var{empirical_age_sex_comm_fit_prob_mi_fnr_median_percent}\% [\var{empirical_age_sex_comm_fit_prob_mi_fnr_lower_percent}\% - \var{empirical_age_sex_comm_fit_prob_mi_fnr_upper_percent}\%], \nameref{empirical_age_sex_comm_table}), implying that empirical phylogenetic signatures of multiple infection under-estimate the true infection status of individuals in any single HIV genomic window. It was therefore essential to have whole-genome data from a subset of participants (Fig~\ref{empirical_data_sum}) to estimate false-negative detection rates. Further, informed by the \var{p_unique_MI_obs_0}\% of samples with no multiple subgraph windows, we estimated the false-positive rate to be low (\var{empirical_age_sex_comm_fit_prob_mi_fpr_median_percent}\% [\var{empirical_age_sex_comm_fit_prob_mi_fpr_lower_percent}\% - \var{empirical_age_sex_comm_fit_prob_mi_fpr_upper_percent}\%]). However, we note that even a low absolute rate will likely give rise to spurious multiple subgraph observations in a large sample size, which warrants consideration in our statistical framework.\par

In this model, the estimated prevalence of multiple infections in the study sample was \var{empirical_age_sex_comm_fit_prob_mi_median_percent}\% [\var{empirical_age_sex_comm_fit_prob_mi_lower_percent}\% - \var{empirical_age_sex_comm_fit_prob_mi_upper_percent}\%] (\nameref{empirical_age_sex_comm_table}). Relaxing our minor subgraph frequency-based filtering step resulted in only a slightly higher prevalence of multiple infections in the study sample (\var{empirical_age_sex_comm_incl_minor_fit_prob_mi_median_percent}\% [\var{empirical_age_sex_comm_incl_minor_fit_prob_mi_lower_percent}\% - \var{empirical_age_sex_comm_incl_minor_fit_prob_mi_upper_percent}\%], \nameref{empirical_age_sex_comm_incl_minor_table}). When considering only genome windows spanning the p24 and gp41 regions as in previous studies (e.g. \cite{redd2011, redd2012, redd2014}), we were unable to estimate $\sigma_{ind}$ (Eq~\ref{eq:base_priors}) with suitably high effective sample size (ESS) values as there were at most two regions of data for each sample. We therefore fixed $\sigma_{ind} = 0.7$ based on the whole-genome analysis (\nameref{empirical_age_sex_comm_table}) and found that that the sample prevalence of multiple infections based on p24 and gp41 was considerably lower as compared to the whole-genome analysis (\var{empirical_age_sex_comm_amplicon_fit_prob_mi_median_percent}\% [\var{empirical_age_sex_comm_amplicon_fit_prob_mi_lower_percent}\% - \var{empirical_age_sex_comm_amplicon_fit_prob_mi_upper_percent}\%],  \nameref{empirical_age_sex_comm_amplicon_table}), highlighting the utility of incorporating whole-genome data into our inference. Finally, after adjusting for slight  biases in the availability of sequence data among viremic participants (Table \ref{seq_table}) using post-stratification based on age, sex, and community type ( \nameref{poststrat_table}), the prevalence of multiple infections among viremic PLHIV in the RCCS was estimated to be slightly lower than the prevalence in the sequenced sample (\var{empirical_age_sex_comm_fit_wprob_mi_median_percent}\% [\var{empirical_age_sex_comm_fit_wprob_mi_lower_percent}\% - \var{empirical_age_sex_comm_fit_wprob_mi_upper_percent}\%], Fig~\ref{empirical_age_sex_comm_fit}C). \par

We next used our model to identify individuals with likely multiple infection based on their within-host phylogenetic trees and our modeling framework. Classification was based on the inferred, posterior multiple infection probabilities, and therefore our model-based approach accounted for individual-level factors associated with sequencing success and population-level false-postive and false-negative rates (Eq~\ref{eq:ind_prob_mi}). We determined a binary classification cut-off above which individuals were classified as having a likely multiple infection such that the total number of identified individuals was consistent with the estimated prevalence in the sample, which resulted in a cut-off of \var{empirical_age_sex_comm_fit_ind_prob_mi_empirical_median_percent}\%. Using this threshold, we estimated there were \var{empirical_age_sex_comm_fit_ind_prob_mi_ge_empirical_median} individuals with a likely multiple infection (Fig~\ref{empirical_age_sex_comm_fit}B).

% -- FIGURE 3 --%
\begin{figure}[!ht]
\ifthenelse{\boolean{includefigs}}{
\centering
\includegraphics[width=0.75\textwidth]{../figures/empirical_full_fit.pdf}
}{}
\caption{{\bf Individual-level estimates and population-level characteristics of HIV multiple infection in people with viremic HIV in the Rakai Community Cohort Study, \protect\var{min_survey_year}-\protect\var{max_survey_year}.}
(A) Estimated posterior probability of multiple infection for each participant. Confidence bounds represent the 95\% highest posterior density. Participants with at least one multiple subgraph window are shown in purple. (B) Number of participants with multiple infection as a function of the threshold used to dichotomize the probability of multiple infection. Central estimate uses the median estimated prevalence of multiple infections and shading uses 95\% and 50\% HPD. Horizontal dotted line plotted at the number of participants needed to match the estimated population prevalence of multiple infection. (C) Posterior distribution of the prevalence of multiple infections among viremic participants in the RCCS after accounting for sampling biases. Bins are shaded according to the 95\% and 50\% HPD. Histogram width is calculated such that there are approximately 50 bins over the range of the plotted values. }
\label{empirical_age_sex_comm_fit}
\end{figure}

\subsection{Risk factors of HIV multiple infection}
In African contexts, HIV infection risk varies at the individual-level, such as by age, gender, sexual behaviour and circumcision status, and  at the community-level \cite{chang2016, grabowski2017, popoola2024}. We therefore next aimed to characterize individual and population-level  risk factors for multiple infection with HIV. First, given the significantly higher prevalence of HIV and viremic HIV in Lake Victoria fishing communities~\cite{chang2016, brizzi2024}, we investigated whether participants with viremic HIV in these communities had increased risk of multiple infection as compared to participants with viremic HIV in inland communities. Using the model described above with age, sex, and community type as predictors of the probability of multiple infection and accounting for sequencing biases through poststratificaiton we calculated the prevalence of multiple infections among viremic PLHIV in fishing and inland communities and found that multiple infections in fishing communities were \var{empirical_age_sex_comm_fit_pr_mi_comm_type == 'fishing':comm_type=='inland'_median} times (95\% HPD \var{empirical_age_sex_comm_fit_pr_mi_comm_type == 'fishing':comm_type=='inland'_lower} - \var{empirical_age_sex_comm_fit_pr_mi_comm_type == 'fishing':comm_type=='inland'_upper})-times more frequent than in inland communities (with posterior median [95\% HPD] prevalence of multiple infection of \var{empirical_age_sex_comm_fit_wprob_mi_comm_type == 'fishing'_median_percent}\% [\var{empirical_age_sex_comm_fit_wprob_mi_comm_type == 'fishing'_lower_percent}\% - \var{empirical_age_sex_comm_fit_wprob_mi_comm_type == 'fishing'_upper_percent}\%]) and \var{empirical_age_sex_comm_fit_wprob_mi_comm_type == 'inland'_median_percent}\% [\var{empirical_age_sex_comm_fit_wprob_mi_comm_type == 'inland'_lower_percent}\% - \var{empirical_age_sex_comm_fit_wprob_mi_comm_type == 'inland'_upper_percent}\%] respectively, Fig~\ref{empirical_mi_predictors}A and \nameref{empirical_age_sex_comm_table}). The estimated prevalence ratio for HIV multiple infection was therefore broadly comparable to the risk ratio of HIV prevalence and viremia in fishing as compared to inland communities (2.5-3)\cite{chang2016, brizzi2024}, consistent with the expectation that the risk of superinfection acquisition scales with the population prevalence of viremic HIV. Because participants from fishing communities are oversampled in our sequence data (\ref{seq_table}, \ref{poststrat_table}), this also explains the lower estimated prevalence of multiple infections in the population as compared to the sample. \par

We additionally incorporated a binary feature describing the sequencing technology used to generate the deep-sequence data from each participant to assess the extent of technical bias in our inferences. In a univariate analysis, we estimated that multiple infections were less common among participants sequenced using the bait-approach protocol (RR median [95\% HPD]: \var{empirical_seq_fit_multivar_rr_mi_sequencing_technology=='bait_capture':sequencing_technology=='amplicon'_median}  [\var{empirical_seq_fit_multivar_rr_mi_sequencing_technology=='bait_capture':sequencing_technology=='amplicon'_lower} - \var{empirical_seq_fit_multivar_rr_mi_sequencing_technology=='bait_capture':sequencing_technology=='amplicon'_upper}] ,\nameref{empirical_seq_table}). However, \var{p_bait_fishing}\% of bait-capture sequenced participants were residents of fishing communities compared to \var{p_amplicon_fishing}\% of amplicon sequenced participants. Consequently, in a bivariate model with community type, the estimated magnitude of the dependence of multiple infection status on sequencing technology was considerably reduced and no longer considered to be significant at the 95\% level. (multivariate RR median [95\% HPD] \var{empirical_comm_seq_fit_multivar_rr_mi_sequencing_technology=='bait_capture':sequencing_technology=='amplicon'_median} [\var{empirical_comm_seq_fit_multivar_rr_mi_sequencing_technology=='bait_capture':sequencing_technology=='amplicon'_lower} - \var{empirical_comm_seq_fit_multivar_rr_mi_sequencing_technology=='bait_capture':sequencing_technology=='amplicon'_upper}], \nameref{empirical_seq_comm_table}). \par

Participants with HIV in fishing communities also reported having more lifetime sex partners (\nameref{mean_obs_sexpever}), so we next assessed whether the risk of harboring a multiple infection differed by the number of self-reported lifetime sex partners within each of the two community locations. As women tend to under-report their number of sex partners relative to men~\cite{todd2009}, we restricted this analysis to male participants. The number of lifetime sexual partners generally increases with age, and so we standardized responses relative to the age-specific mean number of lifetime sexual partners among participants separately for the inland and fishing communities (\nameref{sexpever_std_curve}). Among \var{empirical_sexpever_men_n_participant_visits} male participants included in this analysis, \var{n_men_par_sexpeverNum} reported an exact number of lifetime sex partners, \var{n_men_par_sexpeverMany} responded they had three or more lifetime partners, and \var{n_men_par_sexpeverMissing} did not provide a response. We imputed ambiguous responses and missing data within our inference framework by assuming responses were missing at random between people with and without multiple infection (\nameref{Materials and methods}). 

In a bivariate model with community type and number of lifetime sexual partners we did not find a statistically significantly higher risk of multiple infection in male participants with more lifetime sexual partners in the context of substantial missing data and 
sampling over potential missing values using age-specific prior distributions. However, we note that the posterior effect size translated into an estimated more than two-fold higher risk of multiple infection between men living with viremic HIV in fishing communities associated with having 30 lifetime sexual partners compared to one lifetime sexual partner (e.g. RR median [95\% HPD] among 25-29 year olds \var{empirical_sexpever_men_fit_rr_fishing2429_30v1_median} [\var{empirical_sexpever_men_fit_rr_fishing2429_30v1_lower} - \var{empirical_sexpever_men_fit_rr_fishing2429_30v1_upper}], Fig~\ref{empirical_mi_predictors}B and \nameref{empirical_sexpever_men_prob_mi} for all age groups). Very similar results were observed using a complete case analysis of the \var{n_men_par_sexpeverNum} men who provided an exact number of lifetime sex partners (\nameref{empirical_sexpever_men_complete_table}). \par

We also performed a comprehensive discovery-based risk factor variable selection analysis over eight additive biological, behavioral and epidemic features, stratifying epidemiological and behavioral variables by community type to account demographic differences between the populations and excluding additional variable interactions. This analysis confirmed residency in fishing communities as a risk factor of multiple infection among sequenced participants, albeit with a wide credible interval, (multivariate RR median [95\% HPD] \var{empirical_var_select_fit_multivar_rr_mi_comm_type=='fishing':comm_type=='inland'_median} [\var{empirical_var_select_fit_multivar_rr_mi_comm_type=='fishing':comm_type=='inland'_lower} - \var{empirical_var_select_fit_multivar_rr_mi_comm_type=='fishing':comm_type=='inland'_upper}]), but did not identify any other variables that were associated with significantly higher or lower risk of multiple infection in our sample (Fig~\ref{empirical_mi_predictors}C and \nameref{empirical_var_select_table}). Specifically, despite the fact that female bar/restaurant workers face a three-fold higher risk of incident HIV \cite{popoola2024} we did not identify an increased risk of multiple infection among female bar/restaurant workers or men who have sex with bar/restaurant workers in either inland or fishing communities. \par

% -- FIGURE 4 --%
\begin{figure}[!ht]
\ifthenelse{\boolean{includefigs}}{
\centering
\includegraphics[width=0.75\textwidth]{../figures/empirical_mi_predictors.pdf}
}{}
\caption{{\bf Risk factors of HIV multiple infection among people with viremic HIV in the Rakai Community cohort Study, \protect\var{min_survey_year}-\protect\var{max_survey_year}.} (A) Posterior distribution of the prevalence of multiple infections stratified by community type, accounting for sampling biases, estimated in a multivariate model (age, sex, and community type) with diffuse priors ($n = $ \protect \var{empirical_age_sex_comm_n_participant_visits}). Bins are shaded according to the 95\% and 50\% highest posterior density (HPD). Histogram width is calculated such that there are approximately 50 bins over the range of plotted values. (B) Predicted risk of multiple infection among men aged 25 to 29 years old as a function of lifetime sex partners and community type estimated in a bivariate model with diffuse priors ($n = $ \protect \var{empirical_sexpever_men_n_participant_visits}). Median of the posterior distribution is plotted as the central estimate and shading represents the 95\% and 50\% HPD. Colors are as in (A). (C) Logistic coefficients for the association between putative risk factors and the probability of harboring a multiple infection estimated with Bayesian shrinkage priors ($n = $ \protect \var{empirical_var_select_n_participant_visits}). Sex and bar/rest. work variable includes female sex and bar/restaurant worker and men who report having sex with female sex and bar/restaurant workers. Median of the posterior distribution is plotted as the central estimate, horizontal bars extend to the 95\% and 50\% HPD. Colors are as in (A).}
\label{empirical_mi_predictors}
\end{figure}

%----------------------------------------------------------------------------------------
%   DISCUSSION
%----------------------------------------------------------------------------------------
\section{Discussion}
In this large-scale study, we assessed the prevalence and risk factors of HIV multiple infection in an East African setting with high HIV burden using population-based pathogen deep-sequence surveillance data. To do this, we developed a Bayesian statistical model to identify multiple infections in deep-sequence phylogenies such as those generated by \textit{phyloscanner} \cite{wymant2017}. Our model incorporates false-negative and false-positive rates for the presence of genetically distinct viral variants and simultaneously estimates individual and population-level probabilities of harboring multiple infection. This framework also allows for the identification of biological and epidemiological risk factors for harboring a multiple infection. In simulation analyses, we demonstrated the ability of the model to generate accurate inferences across a range of parameter values, and fitted the model to \textit{phyloscanner} within-host phylogenies inferred from HIV whole-genome RNA deep-sequence data collected between \var{min_survey_date} and \var{max_survey_date} from \var{n_unique_participants_N_obs_gt_0} viremic participants in the Rakai Community Cohort Study, a population-based open-cohort in southern Uganda. Among viremic participants in this study over the study period, the estimated prevalence of multiple infections was approximately \var{empirical_age_sex_comm_fit_wprob_mi_median_percent_int}\%, reflecting the prevalence of co-circulating multiple infections present at time of sampling. Further, we showed that viremic participants with HIV living in high HIV prevalence fishing communities along Lake Victoria were more than twice as likely to harbor a multiple infection as compared to those living in inland agrarian or trading communities. Among male residents in fishing communities, we estimated that those with more lifetime sex partners can be expected to be more likely to have a multiple infection, although this finding did not reach statistical significance at the 95\% level. \par

This study represents the largest analysis of HIV multiple infections by more than an order of magnitude \cite{yuan2024} and rigorously accounts for partial sequencing success and uncertainty in individual-level estimates when estimating population-level risk of multiple infection. Our model indicated that in the context of incomplete genome coverage, as is common in HIV whole-genome sequencing \cite{bonsall2020}, evidence for multiple infections is expected to be observed in only a subset of genome windows. However, we observed a high rate of false-negatives beyond what is expected due to incomplete sequencing, which may be due to insufficient diversity of infecting variants in some regions of the genome \cite{patino2017} to phylogentically distinguish them. This could potentially be due to recombination between infecting variants prior to sampling \cite{fang2004, streeck2008} such that infecting variants are only genetically distinct in some portions of genome when sampled. The population-based multiple infection prevalence estimates from the data reported here are substantially more precise than previous estimates from this setting as expected given the larger sample size and slightly higher than previous estimates ($n=7/149$ \cite{redd2012}), likely primarily reflecting greater sensitivity of whole-genome sequencing data. Multiple infection among inland community study participants in this study (\var{empirical_age_sex_comm_fit_wprob_mi_comm_type == 'inland'_median_percent}\%) was slightly less prevalent than in this earlier work (pre-2009, 4.7\% \cite{redd2012}), consistent with reductions in HIV incidence over the same time frame \cite{grabowski2017}. Previous studies of female sex workers in urban Uganda and Kenya have estimated the prevalence of multiple infections to be as high as $14-16\%$ in this high-risk demographic based on amplicon deep-sequencing \cite{ronen2013,redd2014}. Here, we do not replicate this finding using self-reported data on sex work or bar/restaurant work in our population-based sampling framework. We expect this is likely due to hesitation to self-report sex work among study participants and study participation bias among sex workers. However, our results are generally consistent with previous findings suggesting multiple infections are less common in African populations as compared to the United States (10-15\% in studies conducted between 1996 and 2010 \cite{pacold2012,wagner2013,wagner2014,wagner2016,vesa2017}), which may reflect the fact that the HIV epidemic in the United States is concentrated among men who have sex with men (MSM) and people who inject drugs (PWID) as opposed to the generalized nature of the epidemic in Africa. Further, as the risk of HIV transmission given exposure is 8-16$\times$ and 3-17$\times$ greater for needle-sharing and anal intercourse, respectively, as compared to vaginal intercourse \cite{patel2014}, the risk of multiple infection acquisition given exposure may also be significantly greater in concentrated epidemics. To date, however, we note that the sample size of HIV multiple infection studies in the United States are relatively small ($<$150 individuals) and there is therefore significant uncertainty in the true underlying prevalence in these settings. \par

Our results add to considerable previous research on increased risk of HIV infection among Lake Victoria fishing communities. Previous studies have shown that overall HIV prevalence and prevalence of viremic HIV in these communities is 2.5-3$\times$ higher than in inland communities \cite{chang2016, brizzi2024}, in part due to migration of PLHIV to these communities \cite{grabowski2020,ratmann2020}. Further, despite a rapid increase in antiretroviral therapy (ART) uptake among residents of fishing communities over the study period \cite{kagaayi2019}, there remains a higher prevalence of people living with viremic, ART-resistant HIV as compared to inland communities \cite{martin2023}. We here show that viremic PLHIV in fishing communities also face a significantly higher burden of HIV multiple infections. We also show that among men in fishing communities, multiple infection risk increases with the number of lifetime sex partners. The precision of this estimate is hindered by a large proportion of qualitative responses to this component of the RCCS survey. These results imply that PLHIV in fishing communities continue to be exposed to viremic partners following initial infection. Public health interventions directed at viremic PLHIV in these communities may therefore not only provide life-saving treatment to these individuals but also reduce opportunities for the generation of novel recombinant forms of HIV which could pose challenges to control efforts through potential generation of more transmissible variants and broadening the antigneic space that potential vaccines need to cover \cite{rambaut2004,corey2010, ritchie2014, kiwanuka2009, shriner2004, song2018}. \par

We expect that our inferential framework may be adaptable to whole-genome deep-sequence phylogenies from other pathogens in which infection is chronic (thereby allowing sufficient time for superinfection to occur). Hepatitis C virus (HCV), which is a chronic viral infection transmitted either sexually or by injection drug use, is a natural extension \cite{cunningham2015}. Among people who inject drugs, the prevalence of HCV mixed infections is estimated to be as high as 39\% \cite{van_de_laar2009}. Our framework has the advantage that it uses data from across the genome and does not require haplotyping of sequencing reads, which has proven to be exceedingly difficult with short-read sequence data \cite{eliseev2020}. Recent work has also attempted to identify multiple infections of \textit{Mycobacterium tuberculosis} (MTB), a chronic bacterial infection canonically of the lungs \cite{cohen2012}. These methods work by either clustering allele frequencies to distinguish within- and between-variant differences \cite{sobkowiak2018,gabbassov2021,sobkowiak2024} or by comparing sampled sequence data to a database of reference strains \cite{anyansi2020}. They therefore require defining circulating genetic diversity \textit{a priori} (which may be challenging in a poorly sampled epidemic) or assume independence between alleles, failing to account for linkage between adjacent genome positions and the evolutionary history giving rise to the observed genetic variation. Multiple infections may also be of interest in acute, high-prevalence infectious diseases. For example, severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) superinfections have been observed by identifying mixed alleles as known lineage-defining sites \cite{dezordi2022,wertheim2022}. \par

While \textit{deep-phyloMI} builds upon previous investigations into HIV multiple infections to provide more rigorous estimates of individual and population level parameters, we do rely on some simplifying assumptions in our framework. First, we only identify multiple infections among viremic participants with available deep-sequence data who were identified as part of a putative transmission network. While we adjust for known sampling biases based on demographic characteristics, there may be residual bias such that our sample is non-representative of the underlying population of viremic PLHIV. Further, because the RCCS did not perform viral load testing on all participants prior to the 2014 survey round we adjust only to the demographic characteristics of viremic PLHIV in the four most recent surveys. Further, we focus on identifying multiple infections only in cross-sectional sequence data. As multiple infections can be transient \cite{ronen2013}, we are unable to identify participants who have been but are not currently multiply infected. It is likely that longitudinal sampling or sequencing of the viral reservoir would identify additional individuals who have been multiply infected. Further, with only a single sample per-individual we were unable to reliably identify factors causally associated with incident multiple infection \cite{savitz2022} and therefore report factors that are associated with prevalent multiple infections. Similarly, in the absence of longitudinal data or data sampled soon after initial infection we are unable to reliably distinguish multiple infections acquired through coinfection and superinfection. However, based on our parametrization of the $k$ parameter within \textit{phyloscanner} and the genetic distance between observed multiple subgraphs, we suspect that the vast majority of identified multiple infections are due to superinfection with a genetically distinct viral genotype. More liberal values of $k$ would increase the sensitivity of our approach to identify closely related viral genotypes (such as those acquired during co-infection) at the expense of an increased rate of false-positives. Further, more liberal values of $k$ would be appropriate in settings with less circulating HIV genetic diversity as compared to our study site \cite{kim2024}. \par

HIV multiple infections complicate global control efforts by fueling the generation of genetic diversity \cite{ramirez2008}, worsening clinical outcomes \cite{gottlieb2004, smith2004}, and increasing viral load \cite{smith2004, ronen2013, janes2015}. Here we developed a robust inference framework to identify multiple infections in deep-sequence data and assess the role of epidemiological risk factors, such as living in high burden communities, in harboring multiple infections. This work will inform interventions aimed at preventing the acquisition of HIV superinfections and efforts to model the role iof multiple infections in the dynamics and evolution of HIV. 


%----------------------------------------------------------------------------------------
%   ACKNOWLEDGEMENTS
%----------------------------------------------------------------------------------------
\section*{Acknowledgments}
We thank the participants of the Rakai Community Cohort Study for making this research possible. Further, we thank all Rakai Health Sciences Program staff and all members of the PANGEA-HIV consortium. We thank Dr. Chris Wymant, PhD (Pandemic Sciences Institute, University of Oxford) for insightful discussions about Bayesian modeling of HIV multiple infections and helpful comments on this manuscript. We thank Zhi Ling (Saw Swee Hock School of Public Health, National University of Singapore) for advice on enforcing sum-to-zero constraints in the context or horseshoe-type shrinkage priors. Computational resources were provided through the Imperial College Research Computing Service and the Biomedical Research Computing Cluster at the University of Oxford. This study was supported by the Bill and Melinda Gates Foundation (OPP1084362, INV-007573, INV-035619, INV-060259, INV-075093),the National Institute of Health (NIH) National Institute of Allergy and Infectious Diseases (NIAID, U01AI075115, R01AI087409, U01AI100031, R01AI110324, R01AI114438, K25AI114461, R01AI123002, K01AI125086, R01AI128779, R01AI143333, R21AI145682, R01AI155080, ZIAAI001040), NIH National Institute of Child Health and Development (R01HD050180, R01HD070769, R01HD091003), NIH National Heart, Lung, and Blood Institute (R01HL152813),the Fogarty International Center (D43TW009578, D43TW010557), the Johns Hopkins University Center for AIDS Research (P30AI094189), the U.S. President’s Emergency Plan for AIDS Relief through the Centers for Disease Control and Prevention (NU2GGH000817), and in part by the Division of Intramural Research, NIAID, NIH. MM was supported in part by NIH 1L30AI178824.  
\linespread{1}
\bibliography{references.bib}
\newpage
%----------------------------------------------------------------------------------------
%   SUPPORTING INFORMATION
%----------------------------------------------------------------------------------------
\section*{Supporting information}
% SUPPLEMENTARY FIGURES %
\paragraph*{S1 Fig.}
\label{vl_seq_success}
{\bf Sequencing coverage among samples from \var{n_unique_N_obs_gt_0} Rakai Community Cohort Study participant-visits contributed by viremic people living with HIV with $N^{\text{obs}}_i > 0$, stratified by viral load category and sequencing technology.} (A) Distribution of $N^{\text{obs}}_i$ values for all samples. (B) Number of samples with coverage in each of the \var{n_unique_windows} genome window. 

\paragraph*{S2 Fig.}
\label{empirical_cophenetic_dists}
{\bf Pairwise genetic between unique tips in within-host phylogenetic trees among people with viremic HIV in the Rakai Community Cohort Study,  \protect\var{min_survey_year}-\protect\var{max_survey_year}.} Bins are shaded based on whether tips were assigned to the same subgraph (grey) or different subgraphs (purple), in the case where multiple subgraphs were observed. 

\paragraph*{S3 Fig.}
\label{full_simulation_sensitivity_delta}
{\bf Posterior distribution of parameters in full model fit to simulated data across a range of $\delta$ values.} Rows represent model to fit to simulated data with $\delta=0$ (top row), $\delta=5\%$ (second row), $\delta=10\%$ (third row), and $\delta=20\%$ (bottom row). Posterior distributions bins are shaded according to the 95\% and 50\% highest posterior density. Histogram width is calculated such that there are approximately 50 bins over the range of plotted values. True values are shown as vertical dotted lines. VL = viral load (log\textsubscript{10} copies/mL) normalized to mean = 0 and std. dev = 1. Std. dev. = standard deviation. 

\paragraph*{S4 Fig.}
\label{full_simulation_sensitivity_lambda}
{\bf Posterior distribution of parameters in full model fit to simulated data across a range of $\lambda$ values.} Rows represent model to fit to simulated data with $\lambda=0.1$ (top row), $\lambda=0.2\%$ (second row), $\lambda=0.3\%$ (third row), and $\lambda=0.4\%$ (bottom row). Posterior distributions bins are shaded according to the 95\% and 50\% higheset posterior density. Histogram bin width is calculated such that there are approximately 50 bins over the range of the plotted values. True values are shown as vertical dotted lines. VL = viral load (log\textsubscript{10} copies/mL) normalized to mean = 0 and std. dev = 1. Std. dev. = standard deviation.  

\paragraph*{S5 Fig.}
\label{full_simulation_sensitivity_epsilon}
{\bf Posterior distribution of parameters in full model fit to simulated data across a range of $\epsilon$ values.} Rows represent model to fit to simulated data with $\epsilon=0$ (top row), $\epsilon=0.5\%$ (second row), $\epsilon=1\%$ (third row), and $\epsilon=5\%$ (bottom row). Posterior distributions bins are shaded according to the 95\% and 50\% higheset posterior density. Histogram bin width is calculated such that there are approximately 50 bins over the range of the plotted values. True values are shown as vertical dotted lines. VL = viral load (log\textsubscript{10} copies/mL) normalized to mean = 0 and std. dev = 1. Std. dev. = standard deviation.  

\paragraph*{S6 Fig.}
\label{empirical_age_sex_comm_log_prob_mi}
{\bf Individual-level estimate of HIV multiple infection in people living with viremic HIV in the Rakai Community Cohort Study, \protect\var{min_survey_year}-\protect\var{max_survey_year}.} Estimated posterior log$_{10}$ probability of multiple infection for each participant. Confidence bounds represent the 95\% highest posterior density. Participants with at least one multiple subgraph window are shown in purple. 

\paragraph*{S7 Fig.}
\label{mean_obs_sexpever}
{\bf Mean number of lifetime sex partners stratified by HIV serostatus, sex, community type, and age among \var{n_participant_visits} RCCS participant-visits.} Excludes participant visits in which respondents provided a categorical response (N=\var{n_participant_sexpever_categorical} (\var{p_participant_sexpever_categorical}\%)). 

\paragraph*{S8 Fig.}
\label{sexpever_std_curve}
{\bf Standardization curve used to adjust observed number of lifetime sex partners among men for age-cohort effects.} Includes simple imputation of categorical responses (e.g. ``1-2'' and ``3+'') to 1) the mean value of observed responses of 1 or 2 (``1-2'') within age category and community type and 2) the mean of a lognormal distribution fit to observed responses of $\ge3$ lifetime sex partners within age category and community type. 

\paragraph*{S9 Fig.}
\label{empirical_sexpever_men_prob_mi}
{\bf Posterior estimates of the prevalence of multiple infections, stratified by age category and community type.} Median estimate is plotted as a line and shading represents the 50\% and 95\% highest posterior densities. All age categories share the same coefficient estimates but differ because lifetime sex partner values are standardized to the mean of the observed values within groups defined by sex, age category, and community type.
\newpage

% SUPPLEMENTARY FILES %
\paragraph*{S1 File.}
\label{supp_methods}
{\bf Supplementary methods.}

\paragraph*{S2 File.}
\label{reference_genomes}
{\bf Reference genomes included in the \textit{phyloscanner} analysis.}

\paragraph*{S3 File.}
\label{normalization_constants}
{\bf Normalization constants used to adjust branch lengths in within-host phylogenetic trees.} 

\paragraph*{S4 File.}
\label{model_diagnostics}
{\bf Bayesian model fit diagnostics.}

\paragraph*{S5 File.}
\label{genome_window_sensitivity}
{\bf Sensitivity of results to choice of genome windows.}

% SUPPLEMENTARY TABLES % 
\paragraph*{S1 Table.}
\label{seq_tech_table}
 {\bf Count of participants sequenced using each sequencing protocol.}


\paragraph*{S2 Table.}
\label{epi_table}
{{\bf Characteristics of Rakai Community Cohort Study participant, \protect\var{min_survey_year}-\protect\var{max_survey_year}.} For each participant, includes data from the participant-visit processed with PHSC if applicable or the participant-visit with the highest viral load, using the first visit in the case of ties or for people not living with HIV. Percentages represent the row percentages within each category. Binomial confidence intervals were calculated using the Agresti–Coull method. PHSC = phyloscanner. } 

\paragraph*{S3 Table.}
\label{missing_data_table}
 {{\bf Count of missing values among \var{n_participants} RCCS participants}. For each participant,
includes data from the participant-visit processed with PHSC if applicable or the participant-visit with the
highest viral load, using the first visit in the case of ties or for people not living with HIV. In each category the percentage represents the percentage of all participants or all participants that were viremic and processed with PHSC.}  

\paragraph*{S4 Table.}
\label{poststrat_table}
 {{\bf Viremic participant-visits (2014-2019) and participants with available
phyloscanner output belonging to epidemiological strata in the Rakai Community Cohort Study.}  Epidemiological strata are defined by community type, age category, and sex. As viral load testing was not routinely conducted in earlier study rounds, the viremic participants belonging to each strata were tabulated using only data from the 2014 through 2019 surveys.} 

\paragraph*{S5 Table.}
\label{base_simulation_base_table}
 {\bf Parameter estimates for base model fit to base simulated data.} ESS = effective sample size. HPD = highest posterior density. 


\paragraph*{S6 Table.}
\label{full_simulation_base_table}
 {\bf Parameter estimates for full model fit to full simulated data.} ESS = effective sample size. HPD = highest posterior density. 

\paragraph*{S7 Table.}
\hspace{0.01cm}
\label{full_simulation_full_table}{\bf Parameter estimates for full model fit to full simulated data.} ESS = effective sample size. HPD = highest posterior density. 

\paragraph*{S8 Table.}
\hspace{0.01cm}
\label{ext_simulation_ext_table}
 {\bf Parameter estimates for extended model fit to extended simulated data with epidemiological risk factor of multiple infection.} ESS = effective sample size. HPD = highest posterior density. stz-MVN =
sum-to-zero multivariate Normal distribution.

\paragraph*{S9 Table.}
\hspace{0.01cm}
\label{empirical_age_sex_comm_table}
 {\bf Parameter estimates for full model fit to deep-sequence data from \var{empirical_age_sex_comm_n_participant_visits} RCCS participants living with viremic HIV with age, sex, and community type as putative risk factors for harboring multiple infections} ESS = effective sample size. HPD = highest posterior density. stz-MVN = sum-to-zero multivariate Normal distribution. 

\paragraph*{S10 Table.}
\hspace{0.01cm}
\label{empirical_age_sex_comm_incl_minor_table}
 {\bf Parameter estimates for full model fit to deep-sequence data from \var{empirical_age_sex_comm_amplicon_n_participant_visits} RCCS participants living with viremic HIV with age, sex, and community type as putative risk factors for harboring multiple infections.} Includes data from genome windows spanning the p24 (1427 - 1816) and gp41 (7941 - 8264) regions. ESS = effective sample size. HPD = highest posterior density. stz-MVN = sum-to-zero multivariate Normal distribution. 

\paragraph*{S11 Table.}
\hspace{0.01cm}
\label{empirical_age_sex_comm_amplicon_table}{\bf Parameter estimates for full model fit to deep-sequence data from \var{empirical_age_sex_comm_amplicon_n_participant_visits} RCCS participants living with viremic HIV with age, sex, and community type as putative risk factors for harboring multiple infections.} Includes data from genome windows spanning the p24 (1427 - 1816) and gp41 (7941 - 8264) regions. ESS = effective sample size. HPD = highest posterior density. stz-MVN = sum-to-zero multivariate Normal distribution. 

\paragraph*{S12 Table.}
\hspace{0.01cm}
\label{empirical_seq_table}
 {\bf Parameter estimates for full model fit to deep-sequence data from \var{empirical_seq_n_participant_visits} RCCS participants living with viremic HIV with deep-sequencing protocol as a putative risk factor for harboring multiple infections.} ESS = effective sample size. HPD = highest posterior density. stz-N = sum-to-zero Normal distribution. 

\paragraph*{S13 Table.}
\hspace{0.01cm}
\label{empirical_seq_comm_table}
 {\bf Parameter estimates for full model fit to deep-sequence data from \var{empirical_comm_seq_n_participant_visits} RCCS participants living with viremic HIV with community type and deep-sequencing protocol as putative risk factors for harboring multiple infections} ESS = effective sample size. HPD = highest posterior density. stz-N = sum-to-zero Normal distribution. 


\paragraph*{S14 Table.}
\hspace{0.01cm}
\label{empirical_sexpever_men_table}
 {\bf Parameter estimates for full model fit to deep-sequence data from \var{empirical_sexpever_men_n_participant_visits} men who participated in the RCCS living with viremic HIV with community type and number of lifetime sex partners as putative risk factors for harboring multiple infections adjusted for deep-sequencing protocol..} ESS = effective sample size. HPD = highest posterior density. stz-N = sum-to-zero Normal distribution. 


\paragraph*{S15 Table.}
\hspace{0.01cm}
\label{empirical_sexpever_men_complete_table}
 {\bf Parameter estimates for full model fit to deep-sequence data from \var{empirical_sexpever_men_complete_n_participant_visits} men who participated in the RCCS living with viremic HIV with community type and number of lifetime sex partners as putative risk factors for harboring multiple infections.} Excludes participants with ambiguous or missing data on the number of lifetime sex partners. ESS = effective sample size. HPD = highest posterior density. stz-N = sum-to-zero Normal distribution. 


\paragraph*{S16 Table.}
\hspace{0.01cm}
\label{empirical_var_select_table}
 {\bf Parameter estimates for full model fit to deep-sequence data from \var{empirical_var_select_n_participant_visits} RCCS participants living with viremic HIV with putative risk factors for harboring multiple infection and Bayesian shrinkage priors.} ESS = effective sample size. HPD = highest posterior density. stz-MVN = sum-to-zero multivariate Normal distribution. 

\end{document}

