\documentclass[10pt,letterpaper]{article}
\usepackage[top=1in,left=1in,right=1in, bottom=1in]{geometry}

% amsmath and amssymb packages, useful for mathematical formulas and symbols
\usepackage{amsmath,amssymb}

% cite package, to clean up citations in the main text. Do not remove.
\usepackage{cite}

% Use the PLoS provided BiBTeX style
\bibliographystyle{plos2015}

% textcomp package and marvosym package for additional characters
\usepackage{textcomp,marvosym}

% line numbers
\usepackage[right]{lineno}

% ligatures disabled
\usepackage[nopatch=eqnum]{microtype}
\DisableLigatures[f]{encoding = *, family = * }

% Bold the 'Figure #' in the caption and separate it from the title/caption with a period
% Captions will be left justified
\usepackage[aboveskip=1pt,labelfont=bf,labelsep=period,justification=raggedright,singlelinecheck=off]{caption}
\renewcommand{\figurename}{Fig}
\renewcommand{\thefigure}{S.G.\arabic{figure}}
\renewcommand{\thetable}{S.G.\arabic{table}}
\newcommand{\MI}{M^{\text{\text{obs}}}}


% create \thickcline for thick horizontal lines of variable length
\newlength\savedwidth
\newcommand\thickcline[1]{%
  \noalign{\global\savedwidth\arrayrulewidth\global\arrayrulewidth 2pt}%
  \cline{#1}%
  \noalign{\vskip\arrayrulewidth}%
  \noalign{\global\arrayrulewidth\savedwidth}%
}

% \thickhline command for thick horizontal lines that span the table
\newcommand\thickhline{\noalign{\global\savedwidth\arrayrulewidth\global\arrayrulewidth 2pt}%
\hline
\noalign{\global\arrayrulewidth\savedwidth}}

% Remove this for final submission, do not include graphics
\usepackage{graphicx} % Required for inserting images


% Use nameref to cite supporting information files (see Supporting Information section for more info)
\usepackage{nameref,hyperref}


\title{Quantifying prevalence and risk factors of HIV multiple infection in Uganda from population-based deep-sequence data}

\begin{document}
%TITLE_VPSACE

% Title must be 250 characters or less.
\begin{flushleft}

{\Large\textbf\newline{Quantifying prevalence and risk factors of HIV multiple infection in Uganda from population-based deep-sequence data \\ \medskip \large Supplementary File 1: Supplementary methods} 
%
%
% Please use "sentence case" for title and headings (capitalize only the first word in a title (or heading), the first word in a subtitle (or subheading), and any proper nouns).
}
\newline
% Insert author names, affiliations and corresponding author email (do not include titles, positions, or degrees).
\\
Michael A. Martin\textsuperscript{1,$\dagger$},
Andrea Brizzi\textsuperscript{2},
Xiaoyue Xi\textsuperscript{2,3},
Ronald Moses Galiwango\textsuperscript{4},
Sikhulile Moyo\textsuperscript{5,6},
Deogratius Ssemwanga\textsuperscript{7,8}
Alexandra Blenkinsop\textsuperscript{2},
Andrew D. Redd\textsuperscript{9,10,11},
Lucie Abeler-DÃ¶rner\textsuperscript{12},
Christophe Fraser\textsuperscript{12},
Steven J. Reynolds\textsuperscript{4,9,10},
Thomas C. Quinn\textsuperscript{4,9,10},
Joseph Kagaayi\textsuperscript{4,13},
David Bonsall\textsuperscript{14},
David Serwadda\textsuperscript{4},
Gertrude Nakigozi\textsuperscript{4},
Godfrey Kigozi\textsuperscript{4},
M. Kate Grabowski\textsuperscript{1,4,15,$\dagger$},
Oliver Ratmann\textsuperscript{2,$\dagger$},
with the PANGEA-HIV Consortium and the Rakai Health Sciences Program
\\
\bigskip
\textbf{1} Department of Pathology, Johns Hopkins School of Medicine, Baltimore, MD, USA
\\
\textbf{2} Department of Mathematics, Imperial College London, London, United Kingdom
\\
\textbf{3} Medical Research Council Biostatistics Unit, University of Cambridge, Cambridge, UK \\

\textbf{4} Rakai Health Sciences Program, Kalisizo, Uganda \\

\textbf{5} Botswana Harvard AIDS Institute Partnership, Botswana Harvard HIV Reference Laboratory, Gaborone, Botswana \\

\textbf{6} Harvard T.H. Chan School of Public Health, Boston, MA, USA \\

\textbf{7} Medical Research Council/Uganda Virus Research Institute and London School of Hygiene and Tropical Medicine Uganda Research Unit, Entebbe, Uganda \\

\textbf{8} Uganda Virus Research Institute, Entebbe, Uganda \\

\textbf{9} Department of Medicine, Johns Hopkins School of Medicine, Baltimore, MD, USA
\\
\textbf{10} Division of Intramural Research, National Institute of Allergy and Infectious Diseases, National Institutes of Health, Bethesda, MD, USA
\\
\textbf{11} Institute of Infectious Disease and Molecular Medicine, University of Cape Town, Cape Town, South Africa
\\
\textbf{12} Pandemic Sciences Institute, Nuffield Department of Medicine, University of Oxford, Oxford, UK \\

\textbf{13} Makerere University School of Public Health, Kampala, Uganda \\

\textbf{14} Wellcome Centre for Human Genetics, Nuffield Department of Medicine, University of Oxford, Oxford, UK \\

\textbf{15} Department of Epidemiology, Johns Hopkins Bloomberg School of Public Health, Baltimore, MD, USA \\

\bigskip
% Use the asterisk to denote corresponding authorship and provide email address in note below.
$\dagger$ Corresponding authors mmart108@jhmi.edu, mgrabow2@jhu.edu, oliver.ratmann@imperial.ac.uk 
\end{flushleft}

\section{Supplementary methods}
\subsection{Inference of within-host deep sequence phylogenetic trees}
\subsubsection{Generation of putative transmission networks}
To avoid running phyloscanner simultaneously on all sampled data, we first clustered participants living with HIV into putative transmission networks. Pairwise genetic distances in the form of \% identity between consensus sequences (as generated by shiver) were first calculated in sliding 500 bp windows with a step size of 100 bp to account for recombination and within-host viral evolution following transmission. Pairwise genetic distances excluded sites in which either sequence had a deleted nucleotide or was unsuccessfully genotyped (e.g. an ``N'') or sites in which both sequences were ambiguously genotyped. Exact nucleotide matches were assigned a similarity score of 1 and partial matches based on ambiguous nucleotides in one sequence were assigned a score of 1/2 (biallelelic ambiguous nucleotide) or 1/3 (triallelic). Using genetic distance thresholds calibrated to epidemiologically confirmed transmission pairs within the RCCS~\cite{ratmann2019}, we clustered participants into putative transmission networks based on these pairwise distances. Inferred networks with $>$50 participants were decomposed into smaller networks of variable size by optimizing their modularity~\cite{blondel2008}.\par 

Finally, transmission networks were grouped into sets of sequences by 1) merging small clusters into a single sequence sets of eight participants, 2) incorporating known epidemiologically linked partners (based on RCCS survey data) for each participant in a given transmission network (regardless of genetic distance), and 3) adding 3 participants per network that were highly related to all participants in a given network on average but were not already included in the network. \par
\subsection{Base model accounting for partial sequencing success of infecting variants}
To evaluate the extent to which our base model with no false-positives or false-negatives adequately described the observed data, we calculated an expected value for $\theta_i$ and consequently $(\MI_i | M_i = 1)$ given an observed value for $N^{\text{obs}}_i$:
\begin{align}
   E[\theta_i | N^{\text{obs}}_i, M_i = 1] &= 1 - \sqrt{1-\frac{N^{\text{obs}}_i}{{n^{\text{max}}}}} \\
   E[\MI_i | \theta_i, M_i = 1] &= {n^{\text{max}}}\theta_i^2. 
   \label{eq:phi_MI_expectation}
\end{align}
\linespread{1}
\bibliography{references.bib}
\end{document}